<!DOCTYPE html>
<html>

<!-- Mirrored from mullinslab.microbiol.washington.edu/protocols/tissue_culture_bl3/1725-Pairwise-growth-competition-assay by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:09:35 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    
<title>Pairwise growth competition assay | Mullins Molecular Retrovirology Lab</title>

    
    <link rel="stylesheet" href="style.css" />
    
    <link rel="alternate" type="application/atom+xml" title="Atom feed" href="feed" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-136625084-1"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-136625084-1');
    </script>
</head>
<body id="entry">

<div id="container">

<header>

<h1>Mullins Molecular Retrovirology Lab</h1>

<div class="description">
<ul>
<li>Department of Microbiology</li>
<li>School of Medicine</li>
<li>University of Washington</li>
</ul>

</div>


</header>

<section id="nav">
    <nav>
        
        <ul>
            <li class=""><a href="../../index.html">Home</a></li>
            <li class=""><a href="../../research/index.html">Research</a></li>
            <li class=""><a href="../../publications/index.html">Publications</a></li>
            <li class="here"><a href="../index.html">Protocols</a></li>
            <li class=""><a href="../../people/index.html">People</a></li>
            <li class=""><a href="../../contact/index.html">Contact</a></li>
            <li class=""><a href="../../help/index.html">Help</a></li>
        </ul>
    </nav>
</section>

<section id="links">
    <nav id="quicklinks">
        <h2>Software &amp; Tools</h2>

        <ul>
            <li><a href="http://indra.mullins.microbiol.washington.edu/">Bioinformatics Tools</a></li>
            <li><a href="https://github.com/mullinslab">GitHub</a></li>
            <li><a href="../../hiris/index.html">HIRIS</a></li>
            <li><a href="../../HMA/index.html">HMA Subtyping Kit</a></li>
            <li><a href="../../isdb/index.html">ISDB</a></li>
            <li><a href="../../tcozy/index.html">TCozy</a></li>
            <li><a href="https://viroverse.washington.edu/">Viroverse</a></li>
        </ul>
    </nav>

    <nav id="lablinks">
        <h2>Lab Links</h2>

        <ul>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/pub/doc/">Computing Docs</a></li>
            <li><a href="../../hiris/private/index.html">HIRIS (Private)</a></li>
            <li><a href="https://tcozy.mullins.microbiol.washington.edu/ice-floe/">ICE Floe</a></li>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/git/">Internal Git</a></li>
            <li><a href="../../twiki/index.html">Internal wiki</a></li>
            <li><a href="https://viroverse.washington.edu/viroblast/">Local ViroBLAST</a></li>
            <li><a href="https://redash.mullins.microbiol.washington.edu/">Redash</a></li>
            <li><a href="https://mullinslab.slack.com/">Slack</a></li>
            <li><a href="../../support/index.html">Ticketing system</a></li>
        </ul>
    </nav>

</section>

<div id="cfar"><a href="http://cfar.washington.edu/">
    <img src="../../static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg" width="500" height="157" srcset="/static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg 1x, /static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_1000x314.jpg 2x" alt="University of Washington/Fred Hutch Center for AIDS Research">
</a></div>

<section id="content">
    
<article>
<h1>Pairwise growth competition assay</h1>

<div id="intro">
<p><a href="https://www.jove.com/video/52610/pairwise-growth-competition-assay-for-determining-replication-fitness" title="Pairwise Growth Competition Assay for Determining the Replication Fitness of Human Immunodeficiency Viruses">Pairwise Growth Competition Assay for Determining the Replication
Fitness of Human Immunodeficiency
Viruses</a></p><p><strong>PROTOCOL:</strong></p><p><strong>*For figures and references, please check the paper
(</strong><a href="http://www.jove.com/video/52610"><strong>http://www.jove.com/video/52610</strong></a><strong>)
or the video</strong></p><p>This protocol is <a href="../Mullins-Manocheewa-Pairwise-growth-competition-assay.html">also available as a Word
document</a>.</p><p>**<br>
**</p><p><strong>1)</strong> <strong>Construction of chimeric HIV-1 NL4-3 molecular clones</strong>  </p>
<hr>
<p><strong>1.1)</strong> <strong>Amplify insert DNA fragment</strong></p><p>1.1.1) Design chimeric primers. The 5&#39; halves of both forward and
reverse primers contain an HIV-1 vector sequence, at which the fragment
will be inserted. The 3&#39; half of the primers must contain the end of the
insert sequence (<strong>Figure 2</strong>). Make sure that the chimeric primer
sequence retains the original open reading frames.</p><p>1.1.1.1) Use primers at least 20 bases in length, with a melting
temperature greater than or equal to 60 Â°C, ~50% GC content, and a low
tendency to form primer dimers, heterodimers and/or hairpin structures.
These properties can be assessed using the OligoAnalyzer web tool
(https://www.idtdna.com/analyzer/Applications/OligoAnalyzer/)</p><p>1.1.2) Use PCR<sup>27</sup> and the chimeric primers to amplify insert
DNA (<strong>Figure 2</strong>). For each PCR reaction, use 1X high fidelity buffer,
0.2 mM dNTPs, 1 U high fidelity DNA polymerase, 0.5 ÂµM of forward
chimeric primer, 0.5 ÂµM of reverse chimeric primer, and 1 pg - 10 ng of
DNA sample carrying insert region. Add dH<sub>2</sub>O to a final volume
of 50 Âµl.</p><p>1.1.3) Set thermal cycling steps as follows: Perform an initial DNA
denaturation step at 98 Â°C for 10 sec. Amplify with 30 cycles of DNA
denaturation at 98 Â°C for 10 sec and DNA annealing at 3 Â°C above the
lowest melting temperature of the two primers for 20 sec. Perform a
final extension at 72 Â°C for 10 min. Store PCR products at 4 Â°C.</p><p>1.1.4) Take 5 Âµl of the PCR products from the previous step and run
agarose gel electrophoresis<sup>28</sup>.</p><p>1.1.4.1) Use a 0.7% agarose gel, 1X TAE buffer (40 mM Tris-acetate, 1 mM
EDTA), 0.5 Î¼g/ml ethidium bromide (EtBr) final concentration and 1 kb
ladder as the DNA size marker. Set power source voltage to 5 V/cm
distance between electrodes. Stop the electrophoresis when the loading
dye migrates through about 2/3 of the gel length. Visualize the gel
using a gel documentation system<sup>28</sup>.</p><p>Note: EtBr is a suspected carcinogen and must be properly disposed of,
per institution regulations. Gloves should always be worn when handling
gels containing EtBr. Change to new gloves after finish handling EtBr
containing material and before handling other materials or equipment to
prevent cross contamination.</p><p>1.1.5) If only one DNA band with size corresponding to the desired PCR
product is detected, purify the rest of the PCR product using a
commercial kit such as QIAquick PCR Purification Kit according to the
manufacturer&rsquo;s protocols.</p><p>1.1.5.1) If other, non-specific bands are also present, use the rest of
the PCR product to run preparative gel electrophoresis. Use the same
parameters and conditions specified in step 1.1.4. Ensure that the gel
well is large enough to load ~45 Âµl of PCR products. Cut out the band
of interest and extract the DNA from the gel using QIAquick gel
extraction kit according to the manufacturer&rsquo;s protocols.  </p><p><strong>1.2)</strong> <strong>Introduce the insert fragment into full-length infectious
HIV-1 subtype B vector (pNL4-3)</strong></p><p>1.2.1) Use purified PCR products from step 1.1.5 as PCR primers. Use
pNL4-3VifA<sup>20</sup> as template DNA in one PCR reaction and use
pNL4-3VifB<sup>20</sup> as template in the other reaction (<strong>Figure
2</strong>). For each PCR reaction, use 1X high fidelity buffer, 0.2 mM dNTPs,
2 U high fidelity DNA polymerase, 500 ng of primer DNA and 50 ng of
template DNA in a final volume of 50 Âµl. Set the thermal cycling
parameters to: 98 Â°C for 30 sec, 35 cycles at 98 Â°C for 10 sec, 48 Â°C
for 1 min and 72 Â°C for 10 min, followed by 72 Â°C for 10 min.  </p><p>1.2.2) Add 10 U of <em>Dpn</em>I to 50 Âµl of the PCR reaction and incubate at
37 <sup>o</sup>C for 1 hr to digest the template DNA. Ensure that the
plasmid DNA is isolated from a methylation competent bacterial strain,
e.g., TOP10 chemically competent <em>Escherichia coli</em>.*  </p><p>*</p><p>1.2.3) Use <em>Dpn</em>I digested product to transform competent bacterial
cells. Use heat-shock transformation with TOP10 chemically competent <em>E.
coli</em>, according to the manufacturer&rsquo;s protocol. To select for bacterial
cells containing the recombinant plasmid, use Luria Broth (LB) culture
plates containing 100 mg/L carbenicillin.  </p><p>1.2.3.1) Pick ~10 well-separated colonies and grow each separately in 3
ml LB liquid medium containing 100 mg/L carbenicillin and incubate at 30
Â°C in a shaker overnight.*  </p><p>*</p><p>1.2.3.2) Use QIAprep Spin Miniprep kit to isolate plasmid DNA from the
bacterial liquid culture, according to the manufacturer&rsquo;s protocol.</p><p>1.2.4) Use double restriction digestion<sup>29</sup> to determine
whether the plasmid DNA contains the proper ** insert. Ensure that one
of the restriction sites exists only within the insert region and the
other restriction site exists only once in the HIV-1 vector, outside of
the insert region.</p><p>1.2.4.1) Digest at least 300 ng of plasmid DNA in a 10 Âµl final
reaction volume. Select restriction buffers, incubation temperature and
incubation time according to the manufacturer&rsquo;s protocol of the selected
restriction enzymes. Take 9 Âµl of the digested DNA and run gel
electrophoresis as described in step 1.1.4. Select recombinant plasmids
that have DNA bands of the predicted sizes.  </p><p>1.2.5) Confirm sequence integrity of the recombinant plasmids by Sanger
sequencing. While rare, unwanted mutation(s) can be introduced during
the PCR reactions.</p><p>1.2.5.1) Sequence both strands of the plasmid DNA. Follow instructions
in step 1.1.1.1 to design sequencing primers. In addition, ensure that
the forward and reverse sequencing primers anneal at least 50 bp
upstream and downstream of the insert region in the recombinant plasmid,
respectively.</p><p>1.2.5.2) Submit plasmid DNA and sequencing primers to a commercial DNA
sequencing service provider. Prepare DNA sample and primers as specified
by the service provider.</p><p>1.2.6) Make an endotoxin-free stock of the mutated plasmid DNA using an
Endotoxin free plasmid DNA kit according to manufacturer&rsquo;s protocol.
Prepare at least 1 Âµg of endotoxin-free plasmid DNA for transfection in
the following step.</p><p><strong>1.3)</strong> <strong>Introduce small-scale mutations via site-directed
mutagenesis</strong></p>
<hr>
<p>1.3.1) Design mutagenic primers with overlapping forward and reverse
primers containing the desired mutation(s). Position the base(s) to be
substituted, inserted, or deleted in the middle of the primers, flanked
by 10-15 homologous bases. Follow instructions in step 1.1.1.1.</p><p>1.3.2) Use PCR to synthesize mutant plasmids. For each PCR reaction, use
1X high fidelity buffer, 10 mM dNTPs, 2 U of high fidelity DNA
polymerase, 0.5 ÂµM of forward mutagenic primer, 0.5 ÂµM of reverse
mutagenic primer and 50 ng of chimeric pNL4-3VifB, from step 1.2.6, in a
final volume of 50 Âµl. Set the thermal cycling parameters to: 98 Â°C
for 30 sec, 25 cycles at 98 Â°C for 10 sec, 48 Â°C for 1 min and 72 Â°C
for 10 min, followed by 72 Â°C for 10 min.</p><p>1.3.3) Repeat step 1.2.2 to 1.2.6.</p><p><strong>2)</strong> <strong>Generation of viral stock using transfection</strong></p><p>2.1) Calculate the amount of viral stock desired and plasmid DNA
required. With a viral titer of 10<sup>4</sup> IU/ml or higher, 1.8 ml
of viral stock is sufficient for two sets of growth competition assays,
including monoinfections, each done in triplicate. For a transfection
done in a 6-well plate, about 1.8 ml supernatant is harvested per well.
One Âµg of plasmid DNA is needed for each transfection done in a 6-well
plate.  </p><p>2.2) For each well of a 6-well plate, prepare 100 Âµl of transfection
mixture, e.g., consisting of 1 Âµl X-tremeGENE 9 transfection reagent
(or comparable product), 1 Âµg of plasmid DNA and serum-free DMEM.</p><p>2.2.1) Determine the volume of plasmid DNA needed, using 1 Âµg plasmid
DNA per well. Ensure that the final concentration of the plasmid DNA is
at least 50 ng/Âµl.  </p><p>2.2.2) Determine how much serum-free medium (DMEM) is needed per well
using the formula: Total volume of DMEM in Âµl = 100 ÂµL - DNA volume in
Âµl.  </p><p>2.3) Add 10<sup>6</sup> HEK 293T-17 (ATCC) cells/well in 2 ml of
propagation medium (DMEM + 10% fetal bovine serum (FBS)) into a 6-well
plate. Incubate for 1 hr at 37 <sup>o</sup>C in a 5% CO<sub>2</sub>
atmosphere. Seed as many wells as needed (determined in step 2.1).  </p><p>2.4) To prepare the transfection mixture, aliquot the appropriate volume
of serum-free DMEM, as calculated above, into a 1.8 ml polypropylene
microcentrifuge tube, and then add the transfection reagent.  </p><p>2.4.1) Pipette reagent directly into the media solution, do not add it
to the plastic surface of the microcentrifuge tube. Add plasmid DNA
last. Pipet up and down gently to mix the solution. Incubate for 15 min
at room temperature (15 <sup>o</sup>C to 25 <sup>o</sup>C) to allow the
formation of transfection complexes.  </p><p>2.4.2) Add the mixture in a drop-wise manner to cells seeded in the
6-well plate. Gently shake or swirl the wells to ensure even
distribution of transfection complexes.  </p><p>2.4.3) Seal plates with plastic wrap.  </p><p>2.5) Incubate cultures at 37 <sup>o</sup>C in a 5% CO<sub>2</sub>
atmosphere for 48
hr.<span id="Wash_optional"></span><span id="Harvest_day_2"></span>  </p><p>2.6) Use a pipette to carefully collect and transfer supernatant to a 15
ml tube through a 0.22 Âµm filter top<em>.</em>  </p><p>2.7) Use pipette to transfer 250 Âµl or more of the filtered supernatant
to 1.8 ml microfuge tubes with rubber gaskets in the lids.  </p><p>2.8) Store filtered supernatants at -80 <sup>o</sup>C until use.**</p>
<hr>
<p><strong>3)</strong> <strong>Determine infectious titer of viral stocks on peripheral blood
mononuclear cells (PBMCs)</strong></p>
<hr>
<p>3.1) Stimulate PBMCs with phytohemagglutinin (PHA). Per one viral stock,
seed 2 x 10<sup>6</sup> PBMCs in complete Iscove&rsquo;s Modified Dulbecco&rsquo;s
Medium (cIMDM; IMDM supplemented with 20 U/ml of human interleukin 2
(hIL-2), 10% fetal bovine serum and 1% penicillin/streptomycin)
supplemented with PHA (1.5 Âµg/ml). Seed PBMCs at 2 x 10<sup>6</sup>
cells/ml. Incubate PBMCs at 37 <sup>o</sup>C in a 5% CO<sub>2</sub> ****
atmosphere **** for 72 hr.</p><p>3.2) Harvest PHA stimulated PBMCs. Transfer non-adherent PHA-PBMCs to a
50 ml conical tube. Spin tube at 228 x g for 10 min. Carefully remove
supernatant without disrupting the cell pellet. Re-suspend the cell
pellet to a final concentration of 2 x 10<sup>5</sup> PBMCs/ml in cIMDM.</p><p>3.3) Seed 2x10<sup>4</sup> PHA stimulated PBMCs/well in 100 Âµl/well
cIMDM in a round bottom 96-well plate.  </p><p>3.4) Make a 1:10 dilution of the viral stock. Then, from the first
diluted stock, make twelve 3-fold serial dilutions in a 96-well master
plate. This dilution scheme is recommended for detecting viral titers in
the range of 10<sup>4</sup> to 10<sup>6</sup> infectious unit (IU) per
ml. For example, add 20 Âµl of virus stock to 180 Âµl media in the 1.5
ml tube. Mix the dilution by pipetting carefully. Then transfer 90 Âµl
of the diluted stock into 180 Âµl media of the first well and mix well
by pipetting. Continue dilution series by transferring 90 Âµl from the
current well to 180 Âµl media in the next well eleven more times.
Increase or decrease the initial dilution if titers higher than
10<sup>6</sup> IU/ml or lower than 10<sup>4</sup> IU/ml are expected,
respectively.  </p><p>3.5) Add 40 Âµl of the serially diluted viral stock from the master
dilution plate to the seeded PBMCs plate (from step 3.3) in
quadruplicate. Incubate plates at 37 <sup>o</sup>C in a 5%
CO<sub>2</sub> **** atmosphere **** for 16-24
hr.<span id="d1_wash_optional"></span>  </p><p>3.6) Carefully remove 100 Âµl supernatant from each well, and replace
with 160 Âµl fresh cIMDM to a total volume of 200 Âµl/well. Incubate
plates at 37 <sup>o</sup>C with 5% CO<sub>2</sub> **** atmosphere (day
1).  </p><p>3.7) On days 4, 7, 10 and 13
<span id="d2_4_7_10_treatment"></span>transfer 100 Âµl supernatant from
each well to 100 Âµl disruption buffer (2 % TritonX-100 in PBS), and
replace with 100 Âµl fresh cIMDM. Store the supernatants at -20
<sup>o</sup>C.</p><p>3.7.1) Keep sampling and adding fresh cIMDM every three days until the
titer stabilizes.</p><p>3.8) Determine the 50% tissue culture infectious dose
(TCID<sub>50</sub>) of the viral stock by p24 ELISA using the day 7 and
13 samples as described in step 4.</p><p>3.8.1) If the TCID<sub>50</sub> obtained from day 13 is clearly higher
than the titer from day 7, the virus stock may need a longer time to
expand. Repeat the p24 ELISA using later samples until the infectious
titers from two sampling time points become stable (or decrease). Stocks
should be selected from samples with the highest titers.  </p><p>4) **ELISA (Enzyme-linked Immunoabsorbant assay) detection of HIV-1 p24
for determining viral infectious titer  </p><p>**Note: The following protocol was developed using p24 antigen capture
plates prepared in our laboratory<sup>30</sup>. Commercial HIV-1 p24
ELISA plate/kits can also be used, following the manufacturer&rsquo;s
protocol.  </p><p>4.1) Prior to working with samples, prepare working stocks of primary
antibody (rabbit anti-HIV-1 SF2 p24 antiserum).</p><p>4.1.1) Thaw p24 antiserum at room temperature (RT).</p><p>4.1.2) Mix 2.5 ml glycerol with 2 ml 10% FBS in phosphate buffer saline
(PBS).</p><p>4.1.3) Add 0.5 ml antiserum and mix.</p><p>4.1.4) Store 1 ml aliquots at -20 <sup>o</sup>C.</p><p>4.2) Thaw samples from step 3.7 in 37 <sup>o</sup>C incubator.</p><p>4.3) Wash the p24 capture plate 5 times with wash buffer (1x PBS with
0.05% Tween-20).  </p><p>4.4) Add 50 Âµl/well of sample diluent (1% bovine serum albumin (BSA),
0.2% Tween-20 in RPMI-1640), then add 50 Âµl of sample to appropriate
wells. Include at least three wells with sample diluent only as mock /
negative controls. Incubate for 2 hr at 37 <sup>o</sup>C or overnight at
4 <sup>o</sup>C.</p><p><span id="Add_primary_Ab"></span>4.5) Prepare the primary antibody
solution fresh before use. Make a 1:2,000 fold dilution of the primary
antibody working stocks using the primary antibody diluent (12% FBS in
RPMI-1640). Ensure to prepare enough for the use of 100 Âµl solution per
each sample/control well in a 96-well plate.</p><p>4.5.1) For example, to make enough solution for one 96-well plate, add 5
Âµl of the primary antibody working stocks to the primary antibody
diluent for a final volume of 10 ml.</p><p>4.6) Wash capture plate 5 times with wash buffer.  </p><p>4.7) Add 100 Âµl of the primary antibody solution to each well. Incubate
for 1 h at 37 <sup>o</sup>C in a 5% CO<sub>2</sub> atmosphere.  </p><p><span id="Add_secondary_Ab"></span>4.8) Prepare the secondary antibody
solution fresh before use. Make a 1:14,400 fold dilution of the
secondary antibody (1 mg/ml Goat anti-rabbit HRP) using the secondary
antibody diluent (7 % FBS, 0.01 % Tween-20 in RPMI-1640). To reduce
pipetting errors, perform a two-step serial dilution. Ensure to prepare
enough for the use of 100 Âµl solution per each sample/control well in a
96-well plate.</p><p>4.8.1) For example, to make enough solution for one 96-well plate, first
add 1 Âµl of the secondary antibody to 99 Âµl of the secondary antibody
diluent. Then add 70 Âµl of first dilution to the secondary antibody
diluent for a final volume of 10 ml.  </p><p>4.9) Wash capture plate 5 times with wash buffer.  </p><p>4.10) Add 100 Âµl of the secondary antibody solution to each well.
Incubate for 1 hr at 37 <sup>o</sup>C in a 5% CO<sub>2</sub>
atmosphere.  </p><p>4.11) Wash plate 5 times with wash buffer.  </p><p>4.12) Add<span id="Add_substrate"></span> 100 Âµl of room temperature
TMB substrate. Incubate 30 min at room temperature in a closed container
to protect from light.  </p><p><span id="Add_stop"></span>4.13) Add 100 Âµl of room temperature stop
solution (1 N H<sub>2</sub>SO<sub>4</sub>).  </p><p>4.14) Read absorbance at 450-650 nm in each well using a microplate
reader. Use the absorbance value to score each well as infected or
uninfected. A well is considered to contain infectious virus if the
absorbance value is at least three times higher than the value read from
mock / negative control wells. Calculate TCID<sub>50</sub> of the viral
stock using the Reed-Meunch method<sup>31</sup>.  </p><p>5) **Establish viral growth kinetics  </p><p>**</p><p><span id="d14_treatment"></span><span id="Suppliments"></span><strong>5.1)</strong>
<strong>Monoinfection</strong></p><p>5.1.1) Seed 3x10<sup>5</sup> PHA-stimulated PBMC/well in 48-well plates
in a total volume of 500 Âµl/well. Keep the culture plates at 37
<sup>o</sup>C in a 5% CO<sub>2</sub> atmosphere until inoculation.  </p><p>5.1.2) For each virus, prepare an inoculum containing 6,000 IU in 2 ml
of cIMDM.  </p><p>5.1.3) Inoculate wells in triplicate by adding 500 Âµl of the inoculum
(1,500 IU) to the seeded cells. The final volume of the infected cell
culture is 1 ml/well and the MOI is 0.005.  </p><p>5.1.4) Aliquot 200 Âµl of the remaining inoculum to each of two 96-well
plates for RNA isolation, one of which is saved as a backup.  </p><p>5.1.5) Incubate cultures at 37 <sup>o</sup>C in a 5% CO<sub>2</sub>
atmosphere for 16-24 h.  </p><p>5.1.6) Wash cultures 16-24 h after inoculation.</p><p>5.1.6.1) Remove and discard 750 Âµl of culture supernatant.  </p><p>5.1.6.2) Add 750 Âµl of fresh cIMDM. Wrap plates in plastic wrap and
spin for 10 minutes at 300 x g. Remove and discard 750 Âµl
supernatant.  </p><p>5.1.6.3) Add 750 Âµl of fresh cIMDM. Incubate at 37 <sup>o</sup>C with a
5% CO<sub>2</sub> atmosphere (day 1)**.  </p><p>**</p><p><span id="Daily_Treatment"></span>5.1.7) Sample cultures daily from day
2 to day 7.  </p><p>5.1.7.1) Transfer 500 Âµl of culture supernatant to a 1.8 ml centrifuge
tube. Spin for 1 min at 3000 x g.  </p><p>5.1.7.2) Transfer 200 Âµl of the cell-free supernatant to the two
96-well sample plates for RNA isolation, again saving one plate as
backup. Store supernatants at -80 <sup>o</sup>C until RNA isolation.  </p><p>5.1.7.3) Add 500 Âµl fresh cIMDM to each culture. Incubate at 37
<sup>o</sup>C in a 5% CO<sub>2</sub> atmosphere.  </p><p>5.1.7.4) Discard cultures into Wescodyne at the end of the experiment.  </p><p>5.1.7.5) Isolate RNA from 200 Âµl of supernatant (use commercial kits
such as QIAamp Viral RNA Mini Kit) following the manufacturer&rsquo;s standard
protocol. For a large number of samples, use the Qiagen QIAxtractor.  </p><p>5.1.7.6) Store RNA samples at -80 <sup>o</sup>C until cDNA synthesis.  </p><p><strong>5.2)</strong> <strong>cDNA synthesis (Reverse transcription)</strong></p><p>5.2.1) For each RNA sample, add 1.2 nmol of dNTP and 1.2 pmol of cDNA
synthesis primer, (5&#39;-GTTGATCCTTTAGGTATCTTTCCACAGC-3&#39;, HXB2 nucleotide
7968 to 7995) to 10 Âµl of viral RNA. Add water to a final volume of 14
Âµl. Flick the tube to mix and spin briefly to collect liquid at the
bottom of the tube.  </p><p>5.2.2) Incubate mixture for 5 min at 65 <sup>o</sup>C, then hold at 4
<sup>o</sup>C until the master mix is prepared.  </p><p>5.2.3) Prepare master mix using 5x first-strand buffer (250 mM Tris-HCl,
pH 8.3, 375 mM KCl, 15 mM MgCl<sub>2</sub>), 5 mM DTT, 120 U of
SuperScriptIII and 240 U of Rnase inhibitor. Add water to final volume
of 10 Âµl.  </p><p>5.2.4) Add 10 Âµl of master mix to RNA mixture, flick to mix and spin to
collect.  </p><p>5.2.5) Incubate mixture for 90 min at 50 <sup>o</sup>C to allow
synthesis of cDNA. Incubate for 15 minutes at 70 <sup>o</sup>C to
inactivate reverse transcriptase. Hold at 4 <sup>o</sup>C as needed.  </p><p>5.2.6) Add 2 U of Rnase H, flick to mix, and then spin to collect.  </p><p>5.2.7) Incubate 20 min at 37 <sup>o</sup>C. Store cDNA at -20
<sup>o</sup>C.  </p><p><strong>5.3)</strong> **cDNA quantitation using qPCR system  </p><p>**</p><p>5.3.1) Prepare a standard dilution series. Do 10-fold serial dilution,
from 3 x 10<sup>6</sup> copies/Âµl down to 30 copies/Âµl, of pNL4-3VifA.
Use distilled water for dilutions. Prepare the standard dilution series
fresh before use or prepare small batches and keep at -20 <sup>o</sup>C.
Do not freeze-thaw the standards more than three times.</p><p>5.3.2) Set up a 96-well qPCR reaction plate. Ensure that each plate
contains at least one well of negative controls, a triplicate of the
standard dilution series, and at least a duplicate of each cDNA sample.</p><p>5.3.3) For each qPCR reaction, use 12.5 Âµl of qPCR master mix, 0.2 ÂµM
probe, 0.8 ÂµM each of the forward and reverse primers and 1 Âµl of cDNA
or the standard dilution series or water/buffer (for the negative
control well). Add water to a final volume of 25 Âµl. The qPCR probe is
light sensitive. Keep it in closed container.</p><p>5.3.3.1) To detect cDNA derived from pNL4-3VifA based molecular clone,
use the VifA primer-probe: VifAB forward primer
(GGTCTGCATACAGGAGAAAGAGACT), VifA reverse primer
(5&#39;-AGGGTCTACTTGTGTGCTATATCTCTTTT-3&#39;) and VifAB probe
(6FAM-5&#39;-CTCCATTCTATGGAGACTC-3&#39;-MGBNFQ). For cDNA derived from
pNL4-3VifB based clone, use the VifB primer-probe: VifAB forward primer,
VifAB probe and VifB reverse primer (5&#39;-CACCTGCGTGCTATACCTTTTCT-3&#39;).  </p><p>5.3.4) Set PCR cycling parameters to 50 <sup>o</sup>C for 2 min, 95
<sup>o</sup>C for 10 min, and 40 cycles of 95 <sup>o</sup>C for 15 sec
and 60 <sup>o</sup>C for 1 min. Consult the manufacturer&rsquo;s support
document for operation of the qPCR machine.</p><p>5.3.5) Calculate a standard curve using data from the triplicate
standard dilution series. Compare amplification data of the cDNA sample
to the standard curve to determine the copy number. Consult the
manufacturer&rsquo;s support document for data processing.  </p><p><strong>5.4)</strong> **Determine viral exponential growth phase  </p><p>**</p><p>5.4.1) Plot viral growth kinetics with the sampling day along the X-axis
and the cDNA copy number along the Y-axis and identify the viral
exponential growth phase, i.e., when viral cDNA copy numbers increase in
an exponential progression.</p><p>5.4.2) Use the GRC web tool
(<a href="http://indra.mullins.microbiol.washington.edu/grc/">http://indra.mullins.microbiol.washington.edu/grc/</a>) to calculate
viral growth rate (<em>g</em>). In this application, the GRC tool accepts cDNA
copy numbers from at least two time points as input, and outputs the
viral growth rate (<em>g</em>). Use only time point data within the exponential
growth phase (see step 5.4.1 above) to obtain accurate growth rates. For
a detailed description of the mathematical model used in the GRC web
tool, see<sup>20</sup>.  </p><p><strong>6)</strong> **Growth Competition Assay  </p><p>**</p><p>6.1) Seed 3 x 10<sup>5</sup> PHA-stimulated PBMCs (or 1 x 10<sup>5</sup>
CEMx174 cells<em>)</em> in 500 Âµl total volume per well in a 48 well
flat-bottomed plate.*  </p><p>*</p><p>6.2) Keep plate in 37 <sup>o</sup>C in a 5% CO<sub>2</sub> atmosphere
until inoculation.  </p><p>6.3) For each virus, prepare 3 ml of inoculum containing 6,000 IU.  </p><p>6.4) Transfer 1.5 ml of each viral inoculum to a sterile tube to create
the dual infection inoculum. Add 500 Âµl of the dual inoculum (1,500 IU)
to 3 x 10<sup>5</sup> cells in a 48-well plate. The final culture volume
is 1 ml/well. Aliquot 200 Âµl of the inoculum to two 96-well plates for
RNA isolation; save one plate as a back up.  </p><p>6.5) Incubate inoculated cells at 37 <sup>o</sup>C with a 5%
CO<sub>2</sub> atmosphere for 16-24 hr.</p><p>6.6) Wash cultures 16-24 h after inoculation.</p><p>6.6.1) Remove and discard 750 Âµl of culture supernatant.  </p><p>6.6.2) Add 750 Âµl of fresh cIMDM. Wrap plates in plastic wrap and spin
for 10 minutes at 300 x g. Remove and discard 750 Âµl supernatant.</p><p>6.6.3) Add 750 Âµl of fresh cIMDM. Incubate at 37 <sup>o</sup>C with a
5% CO<sub>2</sub> atmosphere (day 1)<strong>.</strong></p><p>6.7) Select sampling times to include at least 3 time points within the
exponential growth phase observed in step 5.4.1.  </p><p>6.7.1) For each sampling, follow step 5.1.7.  </p><p>6.8) Perform cDNA synthesis as described in section 5.2.  </p><p>6.9) Determine the viral variant ratio using qPCR.</p><p>6.9.1) Prepare a standard serial dilution series in triplicate, diluting
10-fold in each step from 3 x 10<sup>6</sup> copies/Âµl to 30 copies/Âµl
of pNL4-3VifA.</p><p>6.9.2) Set up a 96-well qPCR reaction plate. Ensure that each plate
contains at least one negative control, the standard dilution series in
triplicate, and duplicates of each cDNA sample.</p><p>6.9.3) For each qPCR reaction, use 12.5 Âµl of qPCR Master Mix, 0.2 ÂµM
probe, 0.8 ÂµM each of the forward and reverse primers and 1 Âµl of cDNA
or the standard dilution series or water/buffer (for the negative
control wells). Add water to a final volume of 25 Âµl. The qPCR probe is
light sensitive, keep it in closed container.</p><p>6.9.3.1) Use the VifA primer-probe to detect signals in the negative
controls and the standard dilution series and with one duplicate of the
cDNA sample. Use the VifB primer-probe with the other duplicate of the
cDNA sample.  </p><p>6.9.4) Set PCR cycling parameters to 50 <sup>o</sup>C for 2 min, then 95
<sup>o</sup>C for 10 min, and then 40 cycles of 95 <sup>o</sup>C for 15
sec and 60 <sup>o</sup>C for 1 min. Consult the manufacturer&rsquo;s support
document for operation of the qPCR machine.</p><p>6.9.5) Calculate the standard curve using amplification data from the
standard dilution series. Compare amplification data of the cDNA samples
to the standard curve to determine copy number. Consult the
manufacturer&rsquo;s support document for data processing.</p><p>6.9.6) Use the GRC web tool
(<a href="http://indra.mullins.microbiol.washington.edu/grc/">http://indra.mullins.microbiol.washington.edu/grc/</a>) to calculate
relative viral fitness (<em>d</em>). The GRC tool accepts cDNA copy numbers or
chromatogram peak heights from at least two time points as the input,
and outputs the net growth rate difference (<em>d</em>) between the two
viruses. While the tool can calculate the net growth rate from two time
point data, it is strongly recommended to input data from three or more
time points. Use only data obtained from time points within the
exponential growth phase (see step 5.4) to obtain accurate growth rates.
For a detailed description of the mathematical model used in the GRC web
tool, see<sup>20</sup>.</p><p>6.10) Determine viral ratios using chromatogram peak-heights</p><p>6.10.1) PCR amplify HIV-1 <em>vif</em> fragments containing the VifAB sequence
tag using VifFwd (5â€²-GAAAGAGACTGGCATTTGGGTCAGGG-3â€²; HXB2 positions
5266-5291) and VifRev primers (5â€²-GTCTTCTGGGGCTTGTTCCATCTGTCC-3â€²;
HXB2 positions 5579-5553).</p><p>6.10.1.1) For each PCR reaction, use 1 Âµl of cDNA, 1X NH<sub>4</sub>
buffer, 1.5 mM MgCl<sub>2</sub>, 0.2 mM dNTP, 2.5 U of Taq Polymerase,
and 0.45 ÂµM of each primer. Add water to a final volume of 50 Âµl.  </p><p>6.10.1.2) Set PCR cycling parameters to 3 cycles at 94 <sup>o</sup>C for
1 min, 55 <sup>o</sup>C for 1 min, and 70 <sup>o</sup>C for 1 min,
followed by 34 cycles at 94 <sup>o</sup>C for 15 sec, 58 <sup>o</sup>C
for 30 sec, and 70 <sup>o</sup>C for 1 min, and then hold at 4
<sup>o</sup>C.  </p><p>6.10.2) Purify PCR products using the QIAquick PCR purification kit
according to the manufacturer&rsquo;s protocol.</p><p>6.10.3) Submit purified PCR products to a DNA sequencing service
provider for Sanger sequencing.</p><p>6.10.4) Check the average read quality score, which should be provided
by the sequencing service. If the average base call accuracy is less
than 85%, redo step 6.10.1 to 6.10.3</p><p>6.10.5) Use the ChromatQuant web tool
(<a href="http://indra.mullins.microbiol.washington.edu/cgi-bin/chromatquant.cgi">http://indra.mullins.microbiol.washington.edu/cgi-bin/chromatquant.cgi</a>)
to calculate viral ratio at each time point. The tool requires the
sequence trace file (*.ab1) and the sequence 5&#39; to the nucleotide site
of interest. The tool measures the peak intensity at the specified site.
The ratio of the peak intensity corresponds to the ration of the two
viruses <strong>(Figure 4B).</strong></p><p>6.10.6) Use the GRC web tool
(<a href="http://indra.mullins.microbiol.washington.edu/grc/">http://indra.mullins.microbiol.washington.edu/grc/</a>) and the recorded
peak intensities as the input to calculate viral relative fitness (<em>d</em>).
See step 6.10.6.<sup>20</sup></p>
</div>



<div class="clear"></div>
</article>

</section>

</div>
</body>
<!-- Mirrored from mullinslab.microbiol.washington.edu/protocols/tissue_culture_bl3/1725-Pairwise-growth-competition-assay by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:09:36 GMT -->
</html>