<!DOCTYPE html>
<html>

<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/1211-Intra-host-changes-in-Kaposi-sarcoma-associated-herpesvirus-genomes-in-Ugandan-adults-with-Kaposi-sarcoma by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:05:25 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    
<title>Intra-host changes in Kaposi sarcoma-associated herpesvirus genomes in Ugandan adults with Kaposi sarcoma. | Mullins Molecular Retrovirology Lab</title>

    
    <link rel="stylesheet" href="style.css" />
    
    <link rel="alternate" type="application/atom+xml" title="Atom feed" href="feed" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-136625084-1"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-136625084-1');
    </script>
</head>
<body id="entry">

<div id="container">

<header>

<h1>Mullins Molecular Retrovirology Lab</h1>

<div class="description">
<ul>
<li>Department of Microbiology</li>
<li>School of Medicine</li>
<li>University of Washington</li>
</ul>

</div>


</header>

<section id="nav">
    <nav>
        
        <ul>
            <li class=""><a href="../index.html">Home</a></li>
            <li class=""><a href="../research/index.html">Research</a></li>
            <li class="here"><a href="index.html">Publications</a><ul><li><a href="supplemental.html">Supplemental Data</a></li></ul></li>
            <li class=""><a href="../protocols/index.html">Protocols</a></li>
            <li class=""><a href="../people/index.html">People</a></li>
            <li class=""><a href="../contact/index.html">Contact</a></li>
            <li class=""><a href="../help/index.html">Help</a></li>
        </ul>
    </nav>
</section>

<section id="links">
    <nav id="quicklinks">
        <h2>Software &amp; Tools</h2>

        <ul>
            <li><a href="http://indra.mullins.microbiol.washington.edu/">Bioinformatics Tools</a></li>
            <li><a href="https://github.com/mullinslab">GitHub</a></li>
            <li><a href="../hiris/index.html">HIRIS</a></li>
            <li><a href="../HMA/index.html">HMA Subtyping Kit</a></li>
            <li><a href="../isdb/index.html">ISDB</a></li>
            <li><a href="../tcozy/index.html">TCozy</a></li>
            <li><a href="https://viroverse.washington.edu/">Viroverse</a></li>
        </ul>
    </nav>

    <nav id="lablinks">
        <h2>Lab Links</h2>

        <ul>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/pub/doc/">Computing Docs</a></li>
            <li><a href="../hiris/private/index.html">HIRIS (Private)</a></li>
            <li><a href="https://tcozy.mullins.microbiol.washington.edu/ice-floe/">ICE Floe</a></li>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/git/">Internal Git</a></li>
            <li><a href="../twiki/index.html">Internal wiki</a></li>
            <li><a href="https://viroverse.washington.edu/viroblast/">Local ViroBLAST</a></li>
            <li><a href="https://redash.mullins.microbiol.washington.edu/">Redash</a></li>
            <li><a href="https://mullinslab.slack.com/">Slack</a></li>
            <li><a href="../support/index.html">Ticketing system</a></li>
        </ul>
    </nav>

</section>

<div id="cfar"><a href="http://cfar.washington.edu/">
    <img src="../static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg" width="500" height="157" srcset="/static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg 1x, /static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_1000x314.jpg 2x" alt="University of Washington/Fred Hutch Center for AIDS Research">
</a></div>

<section id="content">
    
<!-- source file: content/publications/33465147-Intra-host-changes-in-Kaposi-sarcoma-associated-herpesvirus-genomes-in-Ugandan-adults-with-Kaposi-sarcoma.md -->
<article>

<div id="intro">
    <h1>Citation Information</h1>
<p>Santiago JC, Goldman JD, Zhao H, Pankow AP, Okuku F, Schmitt MW, Chen LH, Hill CA, Casper C, Phipps WT, Mullins JI (2021). <span class="title">Intra-host changes in Kaposi sarcoma-associated herpesvirus genomes in Ugandan adults with Kaposi sarcoma.</span> <cite class="journalName">PLoS pathogens</cite>, <span class="journalIssue">17(1)</span>, e1008594. (<a href="http://ncbi.nlm.nih.gov/pubmed/33465147">pubmed</a>) (<a href="https://doi.org/10.1371/journal.ppat.1008594">doi</a>)</p>
</div>


<div id="more">
    <h1>Abstract</h1>
<p>Intra-host tumor virus variants may influence the pathogenesis and treatment responses of some virally-associated cancers. However, the intra-host variability of Kaposi sarcoma-associated herpesvirus (KSHV), the etiologic agent of Kaposi sarcoma (KS), has to date been explored with sequencing technologies that possibly introduce more errors than that which occurs in the viral population, and these studies have only studied variable regions. Here, full-length KSHV genomes in tumors and/or oral swabs from 9 Ugandan adults with HIV-associated KS were characterized. Furthermore, we used deep, short-read sequencing using duplex unique molecular identifiers (dUMI)-random double-stranded oligonucleotides that barcode individual DNA molecules before library amplification. This allowed suppression of PCR and sequencing errors to ~10-9/base as well as afforded accurate determination of KSHV genome numbers sequenced in each sample. KSHV genomes were assembled de novo, and rearrangements observed were confirmed by PCR and Sanger sequencing. 131-kb KSHV genome sequences, excluding major repeat regions, were successfully obtained from 23 clinical specimens, averaging 2.3x104 reads/base. Strikingly, KSHV genomes were virtually identical within individuals at the point mutational level. The intra-host heterogeneity that was observed was confined to tumor-associated KSHV mutations and genome rearrangements, all impacting protein-coding sequences. Although it is unclear whether these changes were important to tumorigenesis or occurred as a result of genomic instability in tumors, similar changes were observed across individuals. These included inactivation of the K8.1 gene in tumors of 3 individuals and retention of a region around the first major internal repeat (IR1) in all instances of genomic deletions and rearrangements. Notably, the same breakpoint junctions were found in distinct tumors within single individuals, suggesting metastatic spread of rearranged KSHV genomes. These findings define KSHV intra-host heterogeneity in vivo with greater precision than has been possible in the past and suggest the possibility that aberrant KSHV genomes may contribute to aspects of KS tumorigenesis. Furthermore, study of KSHV with use of dUMI provides a proof of concept for utilizing this technique for detailed study of other virus populations in vivo.</p>
</div>


<div class="clear"></div>
</article>

</section>

</div>
</body>
<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/1211-Intra-host-changes-in-Kaposi-sarcoma-associated-herpesvirus-genomes-in-Ugandan-adults-with-Kaposi-sarcoma by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:05:25 GMT -->
</html>