<!DOCTYPE html>
<html>

<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/182-Dual-HIV-1-infection-associated-with-rapid-disease-progression by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:07:43 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    
<title>Dual HIV-1 infection associated with rapid disease progression. | Mullins Molecular Retrovirology Lab</title>

    
    <link rel="stylesheet" href="style.css" />
    
    <link rel="alternate" type="application/atom+xml" title="Atom feed" href="feed" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-136625084-1"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-136625084-1');
    </script>
</head>
<body id="entry">

<div id="container">

<header>

<h1>Mullins Molecular Retrovirology Lab</h1>

<div class="description">
<ul>
<li>Department of Microbiology</li>
<li>School of Medicine</li>
<li>University of Washington</li>
</ul>

</div>


</header>

<section id="nav">
    <nav>
        
        <ul>
            <li class=""><a href="../index.html">Home</a></li>
            <li class=""><a href="../research/index.html">Research</a></li>
            <li class="here"><a href="index.html">Publications</a><ul><li><a href="supplemental.html">Supplemental Data</a></li></ul></li>
            <li class=""><a href="../protocols/index.html">Protocols</a></li>
            <li class=""><a href="../people/index.html">People</a></li>
            <li class=""><a href="../contact/index.html">Contact</a></li>
            <li class=""><a href="../help/index.html">Help</a></li>
        </ul>
    </nav>
</section>

<section id="links">
    <nav id="quicklinks">
        <h2>Software &amp; Tools</h2>

        <ul>
            <li><a href="http://indra.mullins.microbiol.washington.edu/">Bioinformatics Tools</a></li>
            <li><a href="https://github.com/mullinslab">GitHub</a></li>
            <li><a href="../hiris/index.html">HIRIS</a></li>
            <li><a href="../HMA/index.html">HMA Subtyping Kit</a></li>
            <li><a href="../isdb/index.html">ISDB</a></li>
            <li><a href="../tcozy/index.html">TCozy</a></li>
            <li><a href="https://viroverse.washington.edu/">Viroverse</a></li>
        </ul>
    </nav>

    <nav id="lablinks">
        <h2>Lab Links</h2>

        <ul>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/pub/doc/">Computing Docs</a></li>
            <li><a href="../hiris/private/index.html">HIRIS (Private)</a></li>
            <li><a href="https://tcozy.mullins.microbiol.washington.edu/ice-floe/">ICE Floe</a></li>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/git/">Internal Git</a></li>
            <li><a href="../twiki/index.html">Internal wiki</a></li>
            <li><a href="https://viroverse.washington.edu/viroblast/">Local ViroBLAST</a></li>
            <li><a href="https://redash.mullins.microbiol.washington.edu/">Redash</a></li>
            <li><a href="https://mullinslab.slack.com/">Slack</a></li>
            <li><a href="../support/index.html">Ticketing system</a></li>
        </ul>
    </nav>

</section>

<div id="cfar"><a href="http://cfar.washington.edu/">
    <img src="../static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg" width="500" height="157" srcset="/static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg 1x, /static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_1000x314.jpg 2x" alt="University of Washington/Fred Hutch Center for AIDS Research">
</a></div>

<section id="content">
    
<!-- source file: content/publications/gottlieb_2004-154.md -->
<article>

<div id="intro">
    <h1>Citation Information</h1>
<p>Gottlieb GS, Nickle DC, Jensen MA, Wong KG, Grobler J, Li F, Liu SL, Rademeyer C, Learn GH, Karim SS, Williamson C, Corey L, Margolick JB, Mullins JI (2004). <span class="title">Dual HIV-1 infection associated with rapid disease progression.</span> <cite class="journalName">Lancet (London, England)</cite>, <span class="journalIssue">363(9409)</span>, 619-22. (<a href="http://ncbi.nlm.nih.gov/pubmed/14987889">pubmed</a>)</p>
</div>


<div id="more">
    <h1>Abstract</h1>
<p>Infection with two strains of HIV-1 has implications for understanding HIV transmission and vaccine development; however, frequency and pathogenic consequences of dual infection are unknown. We assessed 64 patients for dual infection with heteroduplex mobility assay, viral sequencing, and phylogenetic methods. HIV disease outcomes were available in 34 patients. Five of these with AIDS endpoints had dual infection with HIV-1: four were cases of coinfection and one was superinfection. In all five, time from seroconversion to clinical AIDS or to CD4+ T-cell count less than 200 cells per microL was very rapid (&lt;3.4 and &lt;3.1 years, respectively). Our findings should prompt larger studies to assess the effect of dual infection at the population level.</p>
<!-- suppData -->

<h1>Supplemental Data</h1>

<h3>SUPPLEMENTARY DATA: METHODS</h3>
<p><strong>Study Patients and Disease Outcomes:</strong> In total, 64 patients were
analyzed for dual HIV-1 infection. These included patients from the
Multicenter AIDS Cohort Study (MACS) (N=32)<sup><a href="#1">1</a></sup>, the
Seattle Primary Infection Cohort (N=10)<sup><a href="#2">2</a></sup>, and the South
African female sex worker cohort (N=22)<sup><a href="#3">3</a>,<a href="#4">4</a></sup>. Of
these 64 patients, 6 were found to be dually infected, whereas 58 only
had evidence of infection from a single source (i.e. singly infected).
Thirty-four had AIDS disease progression outcome data. Five of these 34
patients with outcome data were dually HIV-1 infected: including the
previously described patient B from the Seattle Primary Infection
Cohort<sup><a href="#5">5</a></sup>, 3 patients (14<sup><a href="#6">6</a></sup>, 21, 31) from
the MACS , and 1 patient (Du151) from the South African female sex
worker cohort<sup><a href="#3">3</a></sup>.  </p><p>The MACS patients included both rapid (N=21) and typical (N=11)
progressors. Rapid disease progressors were defined as those that had an
AIDS defining event<sup><a href="#7">7</a></sup> within 5 years of
seroconversion<sup><a href="#8">8</a></sup> and typical progressors were defined as
those that had an AIDS defining event or were still asymptomatic 5 or
more years after seroconversion<sup><a href="#9">9</a>,<a href="#10">10</a></sup>. Of the
singly infected MACS patients, 7 received no antiretroviral therapy; 16
received intermittent monotherapy, 14 received intermittent dual
nucleoside therapy, and 5 received highly active antiretroviral therapy
(HAART) starting at 9.4, 11.7, 12.0, 12.8, and 14.9 years post
seroconversion.  </p><p>In the Seattle Primary Infection Cohort, 2 of the 10 patients were rapid
progressors; 3 were asymptomatic when starting HAART (all within 5 years
of seroconversion) and 3 were lost to follow up within 5 years of
seroconversion; 1 was asymptomatic and started HAART at 7 years after
seroconversion, and in one case we could not confirm the date of
seroconversion (this patient was excluded from further analysis).  </p><p>In the South African female sex worker cohort, patient Du151, was the
only patient of a total of 22, to progress to AIDS prior to the cohort
being disbanded at 2 years. An additional patient (Du179) from the South
African cohort was found to be dually infected but did not have an AIDS
defining event within the 2 years prior to the cohort being disbanded so
could not be further evaluated (her CD4 count = 394/Î¼l and viral load =
3.1 log copies/ml at 16.1 months post-seroconversion).  </p><p>None of the dually infected patients received HAART prior to onset of
AIDS; MACS patient 14 received intermittent monotherapy. MACS patient 21
received mono- and dual NRTI therapy at 1.8 and 3.3 years post
seroconversion, respectively. Patient B received multiple antiretroviral
agents after developing AIDS. None of the South African patients
received ART. Opportunistic infections were treated with appropriate
therapy.  </p><p>Dates of seroconversion were estimated as the midpoint between the last
HIV seronegative visit and the first seropositive visit. HIV-1 plasma
viral loads were measured using standard RT-PCR or bDNA assays. As there
is no consensus on whether a viral &ldquo;set-point&rdquo; occurs in patients with
rapid disease progression, or the best method of its determination, we
report the average of the first year values. In the MACS cohort as a
whole, the ~1 year post-seroconversion plasma HIV viral load has been
shown to be the best predictor of subsequent disease
progression<sup><a href="#11">11</a></sup>. In the 30 patients from the MACS cohort
for whom viral load data was available, there was a positive correlation
between &ldquo;~1 year plasma viral load&rdquo; and subsequent rate of disease
progression (R<sup>2</sup>=0.30, p=0.0026), suggesting our group was
representative of the cohort as a whole. Although there was a trend
toward higher viral loads in the 5 dually infected patients compared to
the singly infected patients, this was not statistically significant
(mean log<sub>10</sub> copies/ml = 5.0 and 4.5, respectively, p=0.19, by
Kruskal-Wallis test).  </p><p><strong>Statistical Analysis:</strong> Rates of disease progression were measured by
time from seroconversion to a clinical AIDS defining
event<sup><a href="#7">7</a></sup> (n=27) or death (n=5), or persistent CD4+ T-cell
count 200/Î¼l (n=27); a total of 34 patients reached at least one
endpoint, 29 patients did not reach an AIDS endpoint and were censored
at time of initiation of HAART or time of lost to follow up (Figure 2a
and 2b). For Kaplan-Meier survival analysis, we censored patients using
both a very conservative &ldquo;intent to treat&rdquo; analysis in which censored
equaled AIDS (Figure 2a) or CD4+ T-cell count 200/Î¼l (Figure 2b) and a
less conservative censored equaled excluded (data not show). Dual
infection was significantly associated with rapid progression to both
endpoints using both &ldquo;intent to treat&rdquo; and &ldquo;censored equaled excluded&rdquo;
analyses (Wilcoxon test for difference in time to AIDS or CD4+ T-cell
count 200/Î¼l between those with dual and those with single infection, p
= 0.005 and p = 0.0025, respectively, for censored equals AIDS (&ldquo;intent
to treat&rdquo;), Figure 2). There was an increased relative risk (adjusted
for censoring) for dual versus single infection for time to AIDS
(RR=2.9; 95% CI = 1.6-4.6; p=0.001) and time to CD4200/Î¼l (RR=3.6 ; 95%
CI=1.9-6.4; p=0.002). In a sub-analysis of only the MACS patients (N=32;
dual=3, single=29), who represented the majority of those patients who
reached true endpoints, dual infection remained significantly associated
with rapid progression to both endpoints (Wilcoxon test p = 0.002 and p
= 0.0006, respectively). Statistical analysis was carried out using JMP
software (SAS Institute, Cary, NC).  </p><p><strong>Patient Samples, Virus Isolation, HMA, PCR, Cloning and Sequencing:</strong>
Patient samples were obtained approximately every 6 months throughout
the course of infection and were screened for dual infection using a
combination of heteroduplex mobility assay (HMA) and virus
sequencing<sup><a href="#12">12</a>,<a href="#13">13</a></sup>. Samples with slow migrating
heteroduplexes on HMA, which are suggestive of dual infection, were
confirmed to be dually infected by viral sequencing (see below). HIV-1
plasma viral RNA and PBMC viral DNA was isolated as previously
described<sup><a href="#5">5</a>,<a href="#13">13</a></sup>. Nested PCR of the HIV-1 C2-V3
region (South African FSW) or C2-V5 (Seattle PIC and MACS) envelope
(<em>env</em>) region was performed using end-point dilution to avoid template
resampling bias<sup><a href="#14">14</a></sup> as previously
described<sup><a href="#5">5</a>,<a href="#13">13</a></sup>. PCR products were cloned and
sequenced using standard methodology. Multiple HIV-1 <em>env</em> sequences
were analyzed from each sample throughout the course of infection for
each patient.  </p><p><strong>Phylogenetic Analysis:</strong> All sequences were assessed for potential
sample mix-up and contamination as previously
described<sup><a href="#15">15</a></sup>. HIV-1 sequences were aligned with
reference sequences from the Los Alamos National Laboratory HIV Database
(<a href="http://hiv-web.lanl.gov/">http://hiv-web.lanl.gov/</a>) with CLUSTALW<sup><a href="#16">16</a></sup> followed
by manual adjustment using MacClade (version 4)<sup><a href="#17">17</a></sup>.
Pairwise evolutionary nucleotide distances (excluding gaps in the
pairwise alignment) were estimated using both distance based and maximum
likelihood methods. Neighbor-joining and maximum likelihood methods were
used to estimate evolutionary models and phylogenetic trees (PAUP*
v4.0)<sup><a href="#18">18</a></sup>. The criteria used for definition of dual
HIV-1 infection were as follows: 1) HIV sequences from an individual
patient were required to cluster into two distinct monophyletic clades
(excluding recombinants) without known epidemiologic linkage (i.e. no
direct or known transmission history from one patient to another); and
2) These clades were required to be no closer to each other, in a
phylogenetic tree, than multiple random unlinked sequences from the
HIV-1 database; and 3) The mean pairwise nucleotide distance between the
two distinct clades was required to be within the range of pairwise
nucleotide distances generated from randomly selected unlinked sequences
in the HIV-1 database. All HIV-1 sequences from the MACS and Seattle PIC
were subtype B, all HIV-1 sequences from the S. Africa FSW cohort were
subtype C.  </p><p><strong>Determination of HIV-1 Superinfection:</strong> HIV-1
superinfection<sup><a href="#19">19</a>-<a href="#23">23</a></sup> in MACS patient 21 was
determined using a combination of HMA and viral sequence analysis as
described above. HIV-1 plasma viral RNA and viral DNA from peripheral
blood mononuclear cells (PBMC) from 0.27, 0.76, 1.3 and 3.3 years post
seroconversion were analyzed (see figure 1). HIV-1 C2-V5 <em>env</em> region
HMA from the 0.27 and 0.76 year time points demonstrated a viral variant
with a unique length polymorphism that was distinct from that of viral
variants from 1.3 and 3.3 years post-seroconversion suggesting
superinfection. Phylogenetic analysis of 131 HIV-1 C2-V5 <em>env</em> region
sequences from plasma and PBMC confirmed superinfection in that there
was no evidence of the superinfecting strain prior to 1.3 years
post-seroconversion (see figure 1). Furthermore, PCR, using primers
specific for the superinfecting strain showed no evidence of it in
plasma or PBMC prior to 1.3 years post-SC at a limit of detection of 1
copy per Î¼g of PBMC DNA. PCR using primers specific for the initial
strain showed no evidence of it in plasma after 0.8 years post-SC,
however in PBMC the initial infecting strain was detectable at low
levels (11 copies per 10<sup>6</sup> PBMC) at 1.3 years post-SC. Sample
mix-up was excluded by molecular HLA class II typing of the pre- and
post- superinfection samples<sup><a href="#24">24</a></sup>.  </p>
<h3>FIGURE LEGENDS:</h3>
<p><strong>Figure 1.</strong> Phylogenetic tree demonstrating dual HIV-1 infection in
patients B, 14, 31, Du151 and 21. C2-V3 envelope (<em>env</em>) sequences
(265bp) were used to construct a Neighbor-Joining tree using the HKY85
model of evolution (implemented in PAUP* v4.0). Sequences from dually
infected patients are shown in symbols (Note: for clarity only a small
subset of the total of 650 HIV sequences obtained from these 5 patients
are shown). Superinfection of patient 21 was detected at 1.3 years
post-seroconversion; filled stars represent sequences found
pre-superinfection (pre-SI) (0.3 and 0.8 years post-seroconversion) and
open stars represent sequences found post-superinfection (post-SI) (1.3
and 3.3 years post-seroconversion). Outgroup sequences from the Los
Alamos HIV Database are shown as black lines. Patients B, 14, 31, and 21
cluster with HIV-1 subtype B reference sequences, patient Du151 clusters
with HIV-1 subtype C reference sequences. Nucleotide distances are shown
with scale bar.  </p><p><strong>Figure 2- panel a.</strong> Kaplan-Meier curves comparing time from
seroconversion to AIDS/death (n=32) in the dually infected (N=6; 1
dually infected subject was lost to follow-up at 2 years) and the singly
infected patients (n=57). 63 of 64 total patients were included (in 1
patient we could not confirm date of seroconversion). Patients who
started HAART (n=8) or were lost to follow-up (n=23) prior to reaching
an endpoint (AIDS) were censored to those dates, where censor equals
AIDS event. (Note: The unusually steep drop in the single infection
survival curves (in contrast to previously published reports) is due to
censoring of many of the patients at 2 years and that the MACS patients
studied were mostly rapid progressors). P-value shown for Wilcoxon test
of dual versus singly infected patients for time to AIDS/death.  </p><p><strong>Figure 2- panel b.</strong> Kaplan-Meier curves comparing time from
seroconversion to persistent CD4+ T-cells count 200/Î¼l (n=27) in the
dually infected (N=6; 1 dually infected subject was lost to follow-up at
2 years) and the singly infected patients (n=57). Sixty-three of 64
total patients were included (in 1 patient we could not confirm date of
seroconversion). Patients who started HAART (n=7) or were lost to
follow-up (n=29) prior to reaching an endpoint (CD4+ T-cells count
200/Î¼l) were censored to those dates, where censor equals CD4+
T-cells count 200/Î¼l. [Note: The unusually steep drop in the single
infection survival curves (in contrast to previously published reports)
is due to censoring of many of the patients at 2 years and that the MACS
patients studied were mostly rapid progressors.] P-value shown for
Wilcoxon test of dual versus singly infected patients for time to CD4+
T-cells count 200/Î¼l.</p>
<h3>REFERENCES FOR SUPPLEMENTAL DATA:</h3>

<ol>
<li> <span id="1">K</span>aslow RA, Ostrow DG, Detels R, Phair JP, Polk
BF, Rinaldo CR, Jr. The Multicenter AIDS Cohort Study: rationale,
organization, and selected characteristics of the participants. <em>Am
J Epidemiol</em> 1987;<strong>126</strong>(2):310-318.</li>
<li> <span id="2">S</span>chacker T, Collier AC, Hughes J, Shea T, Corey
L. Clinical and epidemiologic features of primary HIV infection.
<em>Ann Intern Med</em> 1996;<strong>125</strong>(4):257-264.</li>
<li> <span id="3">v</span>an Harmelen J, Williamson C, Kim B, et al.
Characterization of full-length HIV type 1 subtype C sequences from
South Africa. <em>AIDS Res Hum Retroviruses</em> 2001;<strong>17</strong>(16):1527-1531.</li>
<li> <span id="4">W</span>illiamson C, Morris L, Maughan MF, et al.
Characterization and selection of HIV-1 subtype C isolates for use
in vaccine development. <em>AIDS Res Hum Retroviruses</em>
2003;<strong>19</strong>(2):133-144.</li>
<li> <span id="5">L</span>iu SL, Schacker T, Musey L, et al. Divergent
patterns of progression to AIDS after infection from the same
source: human immunodeficiency virus type 1 evolution and antiviral
responses. <em>J Virol</em> 1997;<strong>71</strong>(6):4284-4295.</li>
<li> <span id="6">R</span>inaldo CR, Jr., Gupta P, Huang X, et al.
Anti-HIV-1 memory cytotoxic T lymphocyte responses associated with
changes in CD4+ T cell numbers in the progression of HIV-1
infection. <em>AIDS Res Hum Retroviruses</em> 1998;<strong>14</strong>(16):1423-1433.</li>
<li> <span id="7">1</span>993 revised classification system for HIV
infection and expanded surveillance case definition for AIDS among
adolescents and adults. <em>MMWR Recomm Rep</em> 1992;<strong>41</strong>(RR-17):1-19.</li>
<li> <span id="8">P</span>hair J, Jacobson L, Detels R, et al. Acquired
immune deficiency syndrome occurring within 5 years of infection
with human immunodeficiency virus type-1: the Multicenter AIDS
Cohort Study. <em>J Acquir Immune Defic Syndr</em> 1992;<strong>5</strong>(5):490-496.</li>
<li> <span id="9">M</span>uoz A, Sabin CA, Phillips AN. The incubation
period of AIDS. <em>AIDS</em> 1997;<strong>11</strong>:S69-S76.</li>
<li><span id="10">M</span>unoz A, Wang MC, Bass S, et al. Acquired
immunodeficiency syndrome (AIDS)-free time after human
immunodeficiency virus type 1 (HIV-1) seroconversion in homosexual
men. Multicenter AIDS Cohort Study Group. <em>Am J Epidemiol</em>
1989;<strong>130</strong>(3):530-539.</li>
<li><span id="11">L</span>yles RH, Munoz A, Yamashita TE, et al. Natural
history of human immunodeficiency virus type 1 viremia after
seroconversion and proximal to AIDS in a large cohort of homosexual
men. Multicenter AIDS Cohort Study. <em>J Infect Dis</em>
2000;<strong>181</strong>(3):872-880.</li>
<li><span id="12">D</span>elwart EL, Shpaer EG, McCutchan FE, et al.
Genetic relationships determined by a DNA heteroduplex mobility
assay: Analysis of HIV-1 <em>env</em> genes. <em>Science</em>
1993;<strong>262</strong>:1257-1261.</li>
<li><span id="13">S</span>hankarappa R, Margolick JB, Gange SJ, et al.
Consistent viral evolutionary changes associated with the
progression of human immunodeficiency virus type 1 infection. <em>J
Virol</em> 1999;<strong>73</strong>(12):10489-10502.</li>
<li><span id="14">L</span>iu SL, Rodrigo AG, Shankarappa R, et al. HIV
quasispecies and resampling. <em>Science</em> 1996;<strong>273</strong>(5274):415-416.</li>
<li><span id="15">K</span>orber BTM, Learn G, Mullins JI, Hahn BH,
Wolinsky S. Protecting HIV sequence databases. <em>Nature</em>
1995;<strong>378</strong>(6554):242-243.</li>
<li><span id="16">T</span>hompson JD, Higgins DG, Gibson TJ. CLUSTAL W:
improving the sensitivity of progressive multiple sequence alignment
through sequence weighting, position-specific gap penalties and
weight matrix choice. <em>Nucleic Acids Res</em> 1994;<strong>22</strong>(22):4673-4680.</li>
<li><span id="17">M</span>addison WP, Maddison DR. MacClade - Analysis
of Phylogeny and Character Evolution - Version 4. Sunderland, MA:
Sinauer Associates, Inc., 2001.</li>
<li><span id="18">P</span>AUP 4.0: Phylogenetic Analysis Using Parsimony
(And Other Methods) [Macintosh program]. 4.0b2a version.
Sunderland, MA: Sinauer Associates, Inc., 1999.</li>
<li><span id="19">R</span>amos A, Hu DJ, Nguyen L, et al. Intersubtype
human immunodeficiency virus type 1 superinfection following
seroconversion to primary infection in two injection drug users. <em>J
Virol</em> 2002;<strong>76</strong>(15):7444-7452.</li>
<li><span id="20">J</span>ost S, Bernard MC, Kaiser L, et al. A patient
with HIV-1 superinfection. <em>N Engl J Med</em> 2002;<strong>347</strong>(10):731-736.</li>
<li><span id="21">A</span>ltfeld M, Allen TM, Yu XG, et al. HIV-1
superinfection despite broad CD8+ T-cell responses containing
replication of the primary virus. <em>Nature</em>
2002;<strong>420</strong>(6914):434-439.</li>
<li><span id="22">L</span>evy JA. Is HIV superinfection worrisome?
<em>Lancet</em> 2003;<strong>361</strong>(9352):98-99.</li>
<li><span id="23">K</span>oelsch KK, Smith DM, Little SJ, et al. Clade B
HIV-1 superinfection with wild-type virus after primary infection
with drug-resistant clade B virus. <em>AIDS</em> 2003;<strong>17</strong>(7):F11-16.</li>
<li><span id="24">F</span>renkel LM, Mullins JI, Learn GH, et al.
Genetic evaluation of suspected cases of transient HIV-1 infection
of infants. <em>Science</em> 1998;<strong>280</strong>(5366):1073-1077.</li>
</ol>

</div>


<div class="clear"></div>
</article>

</section>

</div>
</body>
<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/182-Dual-HIV-1-infection-associated-with-rapid-disease-progression by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:07:43 GMT -->
</html>