<!DOCTYPE html>
<html>

<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/2595-Variation-Within-Major-Internal-Repeats-of-KSHV-In-Vivo-Virus-Evolution by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:05:25 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    
<title>Variation Within Major Internal Repeats of KSHV In Vivo. Virus Evolution | Mullins Molecular Retrovirology Lab</title>

    
    <link rel="stylesheet" href="style.css" />
    
    <link rel="alternate" type="application/atom+xml" title="Atom feed" href="feed" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-136625084-1"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-136625084-1');
    </script>
</head>
<body id="entry">

<div id="container">

<header>

<h1>Mullins Molecular Retrovirology Lab</h1>

<div class="description">
<ul>
<li>Department of Microbiology</li>
<li>School of Medicine</li>
<li>University of Washington</li>
</ul>

</div>


</header>

<section id="nav">
    <nav>
        
        <ul>
            <li class=""><a href="../index.html">Home</a></li>
            <li class=""><a href="../research/index.html">Research</a></li>
            <li class="here"><a href="index.html">Publications</a><ul><li><a href="supplemental.html">Supplemental Data</a></li></ul></li>
            <li class=""><a href="../protocols/index.html">Protocols</a></li>
            <li class=""><a href="../people/index.html">People</a></li>
            <li class=""><a href="../contact/index.html">Contact</a></li>
            <li class=""><a href="../help/index.html">Help</a></li>
        </ul>
    </nav>
</section>

<section id="links">
    <nav id="quicklinks">
        <h2>Software &amp; Tools</h2>

        <ul>
            <li><a href="http://indra.mullins.microbiol.washington.edu/">Bioinformatics Tools</a></li>
            <li><a href="https://github.com/mullinslab">GitHub</a></li>
            <li><a href="../hiris/index.html">HIRIS</a></li>
            <li><a href="../HMA/index.html">HMA Subtyping Kit</a></li>
            <li><a href="../isdb/index.html">ISDB</a></li>
            <li><a href="../tcozy/index.html">TCozy</a></li>
            <li><a href="https://viroverse.washington.edu/">Viroverse</a></li>
        </ul>
    </nav>

    <nav id="lablinks">
        <h2>Lab Links</h2>

        <ul>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/pub/doc/">Computing Docs</a></li>
            <li><a href="../hiris/private/index.html">HIRIS (Private)</a></li>
            <li><a href="https://tcozy.mullins.microbiol.washington.edu/ice-floe/">ICE Floe</a></li>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/git/">Internal Git</a></li>
            <li><a href="../twiki/index.html">Internal wiki</a></li>
            <li><a href="https://viroverse.washington.edu/viroblast/">Local ViroBLAST</a></li>
            <li><a href="https://redash.mullins.microbiol.washington.edu/">Redash</a></li>
            <li><a href="https://mullinslab.slack.com/">Slack</a></li>
            <li><a href="../support/index.html">Ticketing system</a></li>
        </ul>
    </nav>

</section>

<div id="cfar"><a href="http://cfar.washington.edu/">
    <img src="../static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg" width="500" height="157" srcset="/static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg 1x, /static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_1000x314.jpg 2x" alt="University of Washington/Fred Hutch Center for AIDS Research">
</a></div>

<section id="content">
    
<!-- source file: content/publications/Santiago_2023_VariationKSHV_Repeats.md -->
<article>

<div id="intro">
    <h1>Citation Information</h1>
<p>Santiago, JC, Westfall, DH, Adams, SV, Okuku, F, Phipps, W, Mullins, JI (2023)<span class="title">Variation within major internal repeats of KSHV in vivo</span><cite class="journalName">Virus Evil.</cite> 2023 May 22;9(1):vead034(<a href="http://ncbi.nlm.nih.gov/pubmed/37325087">pubmed</a>) (<a href="https://doi.org/10.1093/ve/vead034">doi</a>)</p>
</div>


<div id="more">
    <h1>Abstract</h1>
<p>Kaposi&rsquo;s sarcoma-associated herpesvirus (KSHV) is the etiologic agent of Kaposi&rsquo;s sarcoma (KS), yet the viral genetic factors that lead to the development of KS in KSHV-infected individuals have not been fully elucidated. Nearly, all previous analyses of KSHV genomic evolution and diversity have excluded the three major internal repeat regions: the two origins of lytic replication, internal repeats 1 and 2 (IR1 and IR2), and the latency-associated nuclear antigen (LANA) repeat domain (LANAr). These regions encode protein domains that are essential to the KSHV infection cycle but have been rarely sequenced due to their extended repetitive nature and high guanine and cytosine (GC) content. The limited data available suggest that their sequences and repeat lengths are more heterogeneous across individuals than in the remainder of the KSHV genome. To assess their diversity, the full-length IR1, IR2, and LANAr sequences, tagged with unique molecular identifiers (UMIs), were obtained by Pacific Biosciences&#39; single-molecule real-time sequencing (SMRT-UMI) from twenty-four tumors and six matching oral swabs from sixteen adults in Uganda with advanced KS. Intra-host single-nucleotide variation involved an average of 0.16 per cent of base positions in the repeat regions compared to a nearly identical average of 0.17 per cent of base positions in the remainder of the genome. Tandem repeat unit (TRU) counts varied by only one from the intra-host consensus in a majority of individuals. Including the TRU indels, the average intra-host pairwise identity was 98.3 per cent for IR1, 99.6 per cent for IR2 and 98.9 per cent for LANAr. More individuals had mismatches and variable TRU counts in IR1 (twelve/sixteen) than in IR2 (two/sixteen). There were no open reading frames in the Kaposin coding sequence inside IR2 in at least fifty-five of ninety-six sequences. In summary, the KSHV major internal repeats, like the rest of the genome in individuals with KS, have low diversity. IR1 was the most variable among the repeats, and no intact Kaposin reading frames were present in IR2 of the majority of genomes sampled.</p>
</div>


<div class="clear"></div>
</article>

</section>

</div>
</body>
<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/2595-Variation-Within-Major-Internal-Repeats-of-KSHV-In-Vivo-Virus-Evolution by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:05:25 GMT -->
</html>