<!DOCTYPE html>
<html>

<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/417-Extensive-intrasubtype-recombination-in-South-African-human-immunodeficiency-virus-type-1-subtype-C-infections by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:07:00 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    
<title>Extensive intrasubtype recombination in South African human immunodeficiency virus type 1 subtype C infections. | Mullins Molecular Retrovirology Lab</title>

    
    <link rel="stylesheet" href="style.css" />
    
    <link rel="alternate" type="application/atom+xml" title="Atom feed" href="feed" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-136625084-1"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-136625084-1');
    </script>
</head>
<body id="entry">

<div id="container">

<header>

<h1>Mullins Molecular Retrovirology Lab</h1>

<div class="description">
<ul>
<li>Department of Microbiology</li>
<li>School of Medicine</li>
<li>University of Washington</li>
</ul>

</div>


</header>

<section id="nav">
    <nav>
        
        <ul>
            <li class=""><a href="../index.html">Home</a></li>
            <li class=""><a href="../research/index.html">Research</a></li>
            <li class="here"><a href="index.html">Publications</a><ul><li><a href="supplemental.html">Supplemental Data</a></li></ul></li>
            <li class=""><a href="../protocols/index.html">Protocols</a></li>
            <li class=""><a href="../people/index.html">People</a></li>
            <li class=""><a href="../contact/index.html">Contact</a></li>
            <li class=""><a href="../help/index.html">Help</a></li>
        </ul>
    </nav>
</section>

<section id="links">
    <nav id="quicklinks">
        <h2>Software &amp; Tools</h2>

        <ul>
            <li><a href="http://indra.mullins.microbiol.washington.edu/">Bioinformatics Tools</a></li>
            <li><a href="https://github.com/mullinslab">GitHub</a></li>
            <li><a href="../hiris/index.html">HIRIS</a></li>
            <li><a href="../HMA/index.html">HMA Subtyping Kit</a></li>
            <li><a href="../isdb/index.html">ISDB</a></li>
            <li><a href="../tcozy/index.html">TCozy</a></li>
            <li><a href="https://viroverse.washington.edu/">Viroverse</a></li>
        </ul>
    </nav>

    <nav id="lablinks">
        <h2>Lab Links</h2>

        <ul>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/pub/doc/">Computing Docs</a></li>
            <li><a href="../hiris/private/index.html">HIRIS (Private)</a></li>
            <li><a href="https://tcozy.mullins.microbiol.washington.edu/ice-floe/">ICE Floe</a></li>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/git/">Internal Git</a></li>
            <li><a href="../twiki/index.html">Internal wiki</a></li>
            <li><a href="https://viroverse.washington.edu/viroblast/">Local ViroBLAST</a></li>
            <li><a href="https://redash.mullins.microbiol.washington.edu/">Redash</a></li>
            <li><a href="https://mullinslab.slack.com/">Slack</a></li>
            <li><a href="../support/index.html">Ticketing system</a></li>
        </ul>
    </nav>

</section>

<div id="cfar"><a href="http://cfar.washington.edu/">
    <img src="../static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg" width="500" height="157" srcset="/static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg 1x, /static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_1000x314.jpg 2x" alt="University of Washington/Fred Hutch Center for AIDS Research">
</a></div>

<section id="content">
    
<!-- source file: content/publications/rousseau_2007/rousseau_2007-190.md -->
<article>

<div id="intro">
    <h1>Citation Information</h1>
<p>Rousseau CM, Learn GH, Bhattacharya T, Nickle DC, Heckerman D, Chetty S, Brander C, Goulder PJ, Walker BD, Kiepiela P, Korber BT, Mullins JI (2007). <span class="title">Extensive intrasubtype recombination in South African human immunodeficiency virus type 1 subtype C infections.</span> <cite class="journalName">Journal of virology</cite>, <span class="journalIssue">81(9)</span>, 4492-500. (<a href="http://ncbi.nlm.nih.gov/pubmed/17314156">pubmed</a>)</p>
</div>


<div id="more">
    <h1>Abstract</h1>
<p>Recombinant human immunodeficiency virus type 1 (HIV-1) strains containing sequences from different viral genetic subtypes (intersubtype) and different lineages from within the same subtype (intrasubtype) have been observed. A consequence of recombination can be the distortion of the phylogenetic signal. Several intersubtype recombinants have been identified; however, less is known about the frequency of intrasubtype recombination. For this study, near-full-length HIV-1 subtype C genomes from 270 individuals were evaluated for the presence of intrasubtype recombination. A sliding window schema (window, 2 kb; step, 385 bp) was used to partition the aligned sequences. The Shimodaira-Hasegawa test detected significant topological incongruence in 99.6% of the comparisons of the maximum-likelihood trees generated from each sequence partition, a result that could be explained by recombination. Using RECOMBINE, we detected significant levels of recombination using five random subsets of the sequences. With a set of 23 topologically consistent sequences used as references, bootscanning followed by the interactive informative site test defined recombination breakpoints. Using two multiple-comparison correction methods, 47% of the sequences showed significant evidence of recombination in both analyses. Estimated evolutionary rates were revised from 0.51%/year (95% confidence interval [CI], 0.39 to 0.53%) with all sequences to 0.46%/year (95% CI, 0.38 to 0.48%) with the putative recombinants removed. The timing of the subtype C epidemic origin was revised from 1961 (95% CI, 1947 to 1962) with all sequences to 1958 (95% CI, 1949 to 1960) with the putative recombinants removed. Thus, intrasubtype recombinants are common within the subtype C epidemic and these impact analyses of HIV-1 evolution.</p>
<!-- suppData -->

<h1>Supplemental Data</h1>
<p><strong>Supplemental Data for Rousseau et al. 2007:</strong></p><p>Supplemental Figure:
Figure S1: <a href="../_file/79ff9/FigureS1.pdf">Maximum-likelihood tree of subtype C
sequences.</a></p><p>Figure S1: Maximum-likelihood tree of the complete set of subtype C
sequences. Outgroup sequences from other subtypes are indicated in
gray.</p><p><a href="rousseau_2007/2398-Supplemental-Tables.html">Supplemental Tables</a></p>
</div>


<div class="clear"></div>
</article>

</section>

</div>
</body>
<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/417-Extensive-intrasubtype-recombination-in-South-African-human-immunodeficiency-virus-type-1-subtype-C-infections by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:07:02 GMT -->
</html>