<!DOCTYPE html>
<html>

<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/537-HLA-class-I-driven-evolution-of-human-immunodeficiency-virus-type-1-subtype-c-proteome-immune-escape-and-viral-load by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:06:47 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    
<title>HLA class I-driven evolution of human immunodeficiency virus type 1 subtype c proteome: immune escape and viral load. | Mullins Molecular Retrovirology Lab</title>

    
    <link rel="stylesheet" href="style.css" />
    
    <link rel="alternate" type="application/atom+xml" title="Atom feed" href="feed" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-136625084-1"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-136625084-1');
    </script>
</head>
<body id="entry">

<div id="container">

<header>

<h1>Mullins Molecular Retrovirology Lab</h1>

<div class="description">
<ul>
<li>Department of Microbiology</li>
<li>School of Medicine</li>
<li>University of Washington</li>
</ul>

</div>


</header>

<section id="nav">
    <nav>
        
        <ul>
            <li class=""><a href="../index.html">Home</a></li>
            <li class=""><a href="../research/index.html">Research</a></li>
            <li class="here"><a href="index.html">Publications</a><ul><li><a href="supplemental.html">Supplemental Data</a></li></ul></li>
            <li class=""><a href="../protocols/index.html">Protocols</a></li>
            <li class=""><a href="../people/index.html">People</a></li>
            <li class=""><a href="../contact/index.html">Contact</a></li>
            <li class=""><a href="../help/index.html">Help</a></li>
        </ul>
    </nav>
</section>

<section id="links">
    <nav id="quicklinks">
        <h2>Software &amp; Tools</h2>

        <ul>
            <li><a href="http://indra.mullins.microbiol.washington.edu/">Bioinformatics Tools</a></li>
            <li><a href="https://github.com/mullinslab">GitHub</a></li>
            <li><a href="../hiris/index.html">HIRIS</a></li>
            <li><a href="../HMA/index.html">HMA Subtyping Kit</a></li>
            <li><a href="../isdb/index.html">ISDB</a></li>
            <li><a href="../tcozy/index.html">TCozy</a></li>
            <li><a href="https://viroverse.washington.edu/">Viroverse</a></li>
        </ul>
    </nav>

    <nav id="lablinks">
        <h2>Lab Links</h2>

        <ul>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/pub/doc/">Computing Docs</a></li>
            <li><a href="../hiris/private/index.html">HIRIS (Private)</a></li>
            <li><a href="https://tcozy.mullins.microbiol.washington.edu/ice-floe/">ICE Floe</a></li>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/git/">Internal Git</a></li>
            <li><a href="../twiki/index.html">Internal wiki</a></li>
            <li><a href="https://viroverse.washington.edu/viroblast/">Local ViroBLAST</a></li>
            <li><a href="https://redash.mullins.microbiol.washington.edu/">Redash</a></li>
            <li><a href="https://mullinslab.slack.com/">Slack</a></li>
            <li><a href="../support/index.html">Ticketing system</a></li>
        </ul>
    </nav>

</section>

<div id="cfar"><a href="http://cfar.washington.edu/">
    <img src="../static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg" width="500" height="157" srcset="/static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg 1x, /static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_1000x314.jpg 2x" alt="University of Washington/Fred Hutch Center for AIDS Research">
</a></div>

<section id="content">
    
<!-- source file: content/publications/rousseau_2008-217.md -->
<article>

<div id="intro">
    <h1>Citation Information</h1>
<p>Rousseau CM, Daniels MG, Carlson JM, Kadie C, Crawford H, Prendergast A, Matthews P, Payne R, Rolland M, Raugi DN, Maust BS, Learn GH, Nickle DC, Coovadia H, Ndung&#39;u T, Frahm N, Brander C, Walker BD, Goulder PJ, Bhattacharya T, Heckerman DE, Korber BT, Mullins JI (2008). <span class="title">HLA class I-driven evolution of human immunodeficiency virus type 1 subtype c proteome: immune escape and viral load.</span> <cite class="journalName">Journal of virology</cite>, <span class="journalIssue">82(13)</span>, 6434-46. (<a href="http://ncbi.nlm.nih.gov/pubmed/18434400">pubmed</a>) (<a href="https://doi.org/10.1128/JVI.02455-07">doi</a>)</p>
</div>


<div id="more">
    <h1>Abstract</h1>
<p>Human immunodeficiency virus type 1 (HIV-1) mutations that confer escape from cytotoxic T-lymphocyte (CTL) recognition can sometimes result in lower viral fitness. These mutations can then revert upon transmission to a new host in the absence of CTL-mediated immune selection pressure restricted by the HLA alleles of the prior host. To identify these potentially critical recognition points on the virus, we assessed HLA-driven viral evolution using three phylogenetic correction methods across full HIV-1 subtype C proteomes from a cohort of 261 South Africans and identified amino acids conferring either susceptibility or resistance to CTLs. A total of 558 CTL-susceptible and -resistant HLA-amino acid associations were identified and organized into 310 immunological sets (groups of individual associations related to a single HLA/epitope combination). Mutations away from seven susceptible residues, including four in Gag, were associated with lower plasma viral-RNA loads (q &lt; 0.2 [where q is the expected false-discovery rate]) in individuals with the corresponding HLA alleles. The ratio of susceptible to resistant residues among those without the corresponding HLA alleles varied in the order Vpr &gt; Gag &gt; Rev &gt; Pol &gt; Nef &gt; Vif &gt; Tat &gt; Env &gt; Vpu (Fisher&rsquo;s exact test; P &lt; or = 0.0009 for each comparison), suggesting the same ranking of fitness costs by genes associated with CTL escape. Significantly more HLA-B (chi(2); P = 3.59 x 10(-5)) and HLA-C (chi(2); P = 4.71 x 10(-6)) alleles were associated with amino acid changes than HLA-A, highlighting their importance in driving viral evolution. In conclusion, specific HIV-1 residues (enriched in Vpr, Gag, and Rev) and HLA alleles (particularly B and C) confer susceptibility to the CTL response and are likely to be important in the development of vaccines targeted to decrease the viral load.</p>
</div>


<div class="clear"></div>
</article>

</section>

</div>
</body>
<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/537-HLA-class-I-driven-evolution-of-human-immunodeficiency-virus-type-1-subtype-c-proteome-immune-escape-and-viral-load by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:06:47 GMT -->
</html>