<!DOCTYPE html>
<html>

<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/554-Reconstruction-and-function-of-ancestral-center-of-tree-human-immunodeficiency-virus-type-1-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:07:00 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    
<title>Reconstruction and function of ancestral center-of-tree human immunodeficiency virus type 1 proteins. | Mullins Molecular Retrovirology Lab</title>

    
    <link rel="stylesheet" href="style.css" />
    
    <link rel="alternate" type="application/atom+xml" title="Atom feed" href="feed" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-136625084-1"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-136625084-1');
    </script>
</head>
<body id="entry">

<div id="container">

<header>

<h1>Mullins Molecular Retrovirology Lab</h1>

<div class="description">
<ul>
<li>Department of Microbiology</li>
<li>School of Medicine</li>
<li>University of Washington</li>
</ul>

</div>


</header>

<section id="nav">
    <nav>
        
        <ul>
            <li class=""><a href="../index.html">Home</a></li>
            <li class=""><a href="../research/index.html">Research</a></li>
            <li class="here"><a href="index.html">Publications</a><ul><li><a href="supplemental.html">Supplemental Data</a></li></ul></li>
            <li class=""><a href="../protocols/index.html">Protocols</a></li>
            <li class=""><a href="../people/index.html">People</a></li>
            <li class=""><a href="../contact/index.html">Contact</a></li>
            <li class=""><a href="../help/index.html">Help</a></li>
        </ul>
    </nav>
</section>

<section id="links">
    <nav id="quicklinks">
        <h2>Software &amp; Tools</h2>

        <ul>
            <li><a href="http://indra.mullins.microbiol.washington.edu/">Bioinformatics Tools</a></li>
            <li><a href="https://github.com/mullinslab">GitHub</a></li>
            <li><a href="../hiris/index.html">HIRIS</a></li>
            <li><a href="../HMA/index.html">HMA Subtyping Kit</a></li>
            <li><a href="../isdb/index.html">ISDB</a></li>
            <li><a href="../tcozy/index.html">TCozy</a></li>
            <li><a href="https://viroverse.washington.edu/">Viroverse</a></li>
        </ul>
    </nav>

    <nav id="lablinks">
        <h2>Lab Links</h2>

        <ul>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/pub/doc/">Computing Docs</a></li>
            <li><a href="../hiris/private/index.html">HIRIS (Private)</a></li>
            <li><a href="https://tcozy.mullins.microbiol.washington.edu/ice-floe/">ICE Floe</a></li>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/git/">Internal Git</a></li>
            <li><a href="../twiki/index.html">Internal wiki</a></li>
            <li><a href="https://viroverse.washington.edu/viroblast/">Local ViroBLAST</a></li>
            <li><a href="https://redash.mullins.microbiol.washington.edu/">Redash</a></li>
            <li><a href="https://mullinslab.slack.com/">Slack</a></li>
            <li><a href="../support/index.html">Ticketing system</a></li>
        </ul>
    </nav>

</section>

<div id="cfar"><a href="http://cfar.washington.edu/">
    <img src="../static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg" width="500" height="157" srcset="/static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg 1x, /static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_1000x314.jpg 2x" alt="University of Washington/Fred Hutch Center for AIDS Research">
</a></div>

<section id="content">
    
<!-- source file: content/publications/rolland_2007/rolland_2007-198.md -->
<article>

<div id="intro">
    <h1>Citation Information</h1>
<p>Rolland M, Jensen MA, Nickle DC, Yan J, Learn GH, Heath L, Weiner D, Mullins JI (2007). <span class="title">Reconstruction and function of ancestral center-of-tree human immunodeficiency virus type 1 proteins.</span> <cite class="journalName">Journal of virology</cite>, <span class="journalIssue">81(16)</span>, 8507-14. (<a href="http://ncbi.nlm.nih.gov/pubmed/17537854">pubmed</a>)</p>
</div>


<div id="more">
    <h1>Abstract</h1>
<p>The extensive diversity of human immunodeficiency virus type 1 (HIV-1) and its capacity to mutate and escape host immune responses are major challenges for AIDS vaccine development. Ancestral sequences, which minimize the genetic distance to circulating strains, provide an opportunity to design immunogens with the potential to elicit broad recognition of HIV epitopes. We developed a phylogenetics-informed algorithm to reconstruct ancestral HIV sequences, called Center of Tree (COT). COT sequences have potentially significant benefits over isolate-based strategies, as they minimize the evolutionary distances to circulating strains. COT sequences are designed to surmount the potential pitfalls stemming from sampling bias with the consensus method and outlier bias with the most-recent-common-ancestor approach. We computationally derived COT sequences from circulating HIV-1 subtype B sequences for the genes encoding the major viral structural protein (Gag) and two regulatory proteins, Tat and Nef. COT genes were synthesized de novo and expressed in mammalian cells, and the proteins were characterized. COT Gag was shown to generate virus-like particles, while COT Tat transactivated gene expression from the HIV-1 long terminal repeat and COT Nef mediated downregulation of cell surface major histocompatibility complex class I. Thus, retrodicted ancestral COT proteins can retain the biological functions of extant HIV-1 proteins. Additionally, COT proteins were immunogenic, as they elicited antigen-specific cytotoxic T-lymphocyte responses in mice. These data support the utility of the COT approach to create novel and biologically active ancestral proteins as a starting point for studies of the structure, function, and biological fitness of highly variable genes, as well as for the rational design of globally relevant vaccine candidates.</p>
</div>


<div class="clear"></div>
</article>

</section>

</div>
</body>
<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/554-Reconstruction-and-function-of-ancestral-center-of-tree-human-immunodeficiency-virus-type-1-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:07:00 GMT -->
</html>