<!DOCTYPE html>
<html>

<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/584-A-reliable-phenotype-predictor-for-human-immunodeficiency-virus-type-1-subtype-C-based-on-envelope-V3-sequences by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:07:02 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    
<title>A reliable phenotype predictor for human immunodeficiency virus type 1 subtype C based on envelope V3 sequences. | Mullins Molecular Retrovirology Lab</title>

    
    <link rel="stylesheet" href="style.css" />
    
    <link rel="alternate" type="application/atom+xml" title="Atom feed" href="feed" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-136625084-1"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-136625084-1');
    </script>
</head>
<body id="entry">

<div id="container">

<header>

<h1>Mullins Molecular Retrovirology Lab</h1>

<div class="description">
<ul>
<li>Department of Microbiology</li>
<li>School of Medicine</li>
<li>University of Washington</li>
</ul>

</div>


</header>

<section id="nav">
    <nav>
        
        <ul>
            <li class=""><a href="../index.html">Home</a></li>
            <li class=""><a href="../research/index.html">Research</a></li>
            <li class="here"><a href="index.html">Publications</a><ul><li><a href="supplemental.html">Supplemental Data</a></li></ul></li>
            <li class=""><a href="../protocols/index.html">Protocols</a></li>
            <li class=""><a href="../people/index.html">People</a></li>
            <li class=""><a href="../contact/index.html">Contact</a></li>
            <li class=""><a href="../help/index.html">Help</a></li>
        </ul>
    </nav>
</section>

<section id="links">
    <nav id="quicklinks">
        <h2>Software &amp; Tools</h2>

        <ul>
            <li><a href="http://indra.mullins.microbiol.washington.edu/">Bioinformatics Tools</a></li>
            <li><a href="https://github.com/mullinslab">GitHub</a></li>
            <li><a href="../hiris/index.html">HIRIS</a></li>
            <li><a href="../HMA/index.html">HMA Subtyping Kit</a></li>
            <li><a href="../isdb/index.html">ISDB</a></li>
            <li><a href="../tcozy/index.html">TCozy</a></li>
            <li><a href="https://viroverse.washington.edu/">Viroverse</a></li>
        </ul>
    </nav>

    <nav id="lablinks">
        <h2>Lab Links</h2>

        <ul>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/pub/doc/">Computing Docs</a></li>
            <li><a href="../hiris/private/index.html">HIRIS (Private)</a></li>
            <li><a href="https://tcozy.mullins.microbiol.washington.edu/ice-floe/">ICE Floe</a></li>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/git/">Internal Git</a></li>
            <li><a href="../twiki/index.html">Internal wiki</a></li>
            <li><a href="https://viroverse.washington.edu/viroblast/">Local ViroBLAST</a></li>
            <li><a href="https://redash.mullins.microbiol.washington.edu/">Redash</a></li>
            <li><a href="https://mullinslab.slack.com/">Slack</a></li>
            <li><a href="../support/index.html">Ticketing system</a></li>
        </ul>
    </nav>

</section>

<div id="cfar"><a href="http://cfar.washington.edu/">
    <img src="../static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg" width="500" height="157" srcset="/static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg 1x, /static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_1000x314.jpg 2x" alt="University of Washington/Fred Hutch Center for AIDS Research">
</a></div>

<section id="content">
    
<!-- source file: content/publications/jensen_2006/jensen_2006-182.md -->
<article>

<div id="intro">
    <h1>Citation Information</h1>
<p>Jensen MA, Coetzer M, van &rsquo;t Wout AB, Morris L, Mullins JI (2006). <span class="title">A reliable phenotype predictor for human immunodeficiency virus type 1 subtype C based on envelope V3 sequences.</span> <cite class="journalName">Journal of virology</cite>, <span class="journalIssue">80(10)</span>, 4698-704. (<a href="http://ncbi.nlm.nih.gov/pubmed/16641263">pubmed</a>)</p>
</div>


<div id="more">
    <h1>Abstract</h1>
<p>In human immunodeficiency virus type 1 (HIV-1) subtype B infections, the emergence of viruses able to use CXCR4 as a coreceptor is well documented and associated with accelerated CD4 decline and disease progression. However, in HIV-1 subtype C infections, responsible for more than 50% of global infections, CXCR4 usage is less common, even in individuals with advanced disease. A reliable phenotype prediction method based on genetic sequence analysis could provide a rapid and less expensive approach to identify possible CXCR4 variants and thus increase our understanding of subtype C coreceptor usage. For subtype B V3 loop sequences, genotypic predictors have been developed based on position-specific scoring matrices (PSSM). In this study, we apply this methodology to a training set of 279 subtype C sequences of known phenotypes (228 non-syncytium-inducing [NSI] CCR5(+) and 51 SI CXCR4(+) sequences) to derive a C-PSSM predictor. Specificity and sensitivity distributions were estimated by combining data set bootstrapping with leave-one-out cross-validation, with random sampling of single sequences from individuals on each bootstrap iteration. The C-PSSM had an estimated specificity of 94% (confidence interval [CI], 92% to 96%) and a sensitivity of 75% (CI, 68% to 82%), which is significantly more sensitive than predictions based on other methods, including a commonly used method based on the presence of positively charged residues (sensitivity, 47.8%). A specificity of 83% and a sensitivity of 83% were achieved with a validation set of 24 SI and 47 NSI unique subtype C sequences. The C-PSSM performs as well on subtype C V3 loops as existing subtype B-specific methods do on subtype B V3 loops. We present bioinformatic evidence that particular sites may influence coreceptor usage differently, depending on the subtype.</p>
<!-- suppData -->

<h1>Supplemental Data</h1>

<h3>Supplemental Data for Jensen et al.:</h3>

<h3><a href="../_file/61116/JVI02525_05_supplement_022106.xls">All supplemental data as a Microsoft Excel workbook</a></h3>

<ol>
<li> <a href="jensen_2006/2153-Supplemental-Tables.html#T1">Training Set Data</a></li>
<li> <a href="jensen_2006/2153-Supplemental-Tables.html#T2">Validation Analysis</a></li>
<li> <a href="../_file/17a83/NSI.fasta">NSI training set in FASTA format</a>
u[nn] in name line indicates sample from [nn]th infected
individual GenBank acc. (indicated where available)</li>
<li> <a href="../_file/01143/SI.fasta">SI training set in FASTA format</a>
u[nn] in name line indicates sample from [nn]th infected
individual GenBank acc. (indicated where available)</li>
<li> <a href="../_file/8367b/valid.fasta">Validation set in FASTA format</a>
&ldquo;multiple names after &lsquo;&gt;&rsquo; are isolates with identical V3 loops; for
Genbank accessions, see &rdquo;&ldquo;validation analysis&rdquo;&ldquo; sheet&rdquo;</li>
</ol>
<p><a href="jensen_2006/2153-Supplemental-Tables.html">Supplemental tables</a></p>
</div>


<div class="clear"></div>
</article>

</section>

</div>
</body>
<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/584-A-reliable-phenotype-predictor-for-human-immunodeficiency-virus-type-1-subtype-C-based-on-envelope-V3-sequences by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:07:07 GMT -->
</html>