<!DOCTYPE html>
<html>

<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/640-Predicting-demographic-group-structures-based-on-DNA-sequence-data by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:08:03 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    
<title>Predicting demographic group structures based on DNA sequence data. | Mullins Molecular Retrovirology Lab</title>

    
    <link rel="stylesheet" href="style.css" />
    
    <link rel="alternate" type="application/atom+xml" title="Atom feed" href="feed" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-136625084-1"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-136625084-1');
    </script>
</head>
<body id="entry">

<div id="container">

<header>

<h1>Mullins Molecular Retrovirology Lab</h1>

<div class="description">
<ul>
<li>Department of Microbiology</li>
<li>School of Medicine</li>
<li>University of Washington</li>
</ul>

</div>


</header>

<section id="nav">
    <nav>
        
        <ul>
            <li class=""><a href="../index.html">Home</a></li>
            <li class=""><a href="../research/index.html">Research</a></li>
            <li class="here"><a href="index.html">Publications</a><ul><li><a href="supplemental.html">Supplemental Data</a></li></ul></li>
            <li class=""><a href="../protocols/index.html">Protocols</a></li>
            <li class=""><a href="../people/index.html">People</a></li>
            <li class=""><a href="../contact/index.html">Contact</a></li>
            <li class=""><a href="../help/index.html">Help</a></li>
        </ul>
    </nav>
</section>

<section id="links">
    <nav id="quicklinks">
        <h2>Software &amp; Tools</h2>

        <ul>
            <li><a href="http://indra.mullins.microbiol.washington.edu/">Bioinformatics Tools</a></li>
            <li><a href="https://github.com/mullinslab">GitHub</a></li>
            <li><a href="../hiris/index.html">HIRIS</a></li>
            <li><a href="../HMA/index.html">HMA Subtyping Kit</a></li>
            <li><a href="../isdb/index.html">ISDB</a></li>
            <li><a href="../tcozy/index.html">TCozy</a></li>
            <li><a href="https://viroverse.washington.edu/">Viroverse</a></li>
        </ul>
    </nav>

    <nav id="lablinks">
        <h2>Lab Links</h2>

        <ul>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/pub/doc/">Computing Docs</a></li>
            <li><a href="../hiris/private/index.html">HIRIS (Private)</a></li>
            <li><a href="https://tcozy.mullins.microbiol.washington.edu/ice-floe/">ICE Floe</a></li>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/git/">Internal Git</a></li>
            <li><a href="../twiki/index.html">Internal wiki</a></li>
            <li><a href="https://viroverse.washington.edu/viroblast/">Local ViroBLAST</a></li>
            <li><a href="https://redash.mullins.microbiol.washington.edu/">Redash</a></li>
            <li><a href="https://mullinslab.slack.com/">Slack</a></li>
            <li><a href="../support/index.html">Ticketing system</a></li>
        </ul>
    </nav>

</section>

<div id="cfar"><a href="http://cfar.washington.edu/">
    <img src="../static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg" width="500" height="157" srcset="/static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg 1x, /static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_1000x314.jpg 2x" alt="University of Washington/Fred Hutch Center for AIDS Research">
</a></div>

<section id="content">
    
<!-- source file: content/publications/anderson_2003-146.md -->
<article>

<div id="intro">
    <h1>Citation Information</h1>
<p>Anderson JP, Learn GH, Rodrigo AG, He X, Wang Y, Weinstock H, Kalish ML, Robbins KE, Hood L, Mullins JI (2003). <span class="title">Predicting demographic group structures based on DNA sequence data.</span> <cite class="journalName">Molecular biology and evolution</cite>, <span class="journalIssue">20(7)</span>, 1168-80. (<a href="http://ncbi.nlm.nih.gov/pubmed/12777527">pubmed</a>)</p>
</div>


<div id="more">
    <h1>Abstract</h1>
<p>The ability to infer relationships between groups of sequences, either by searching for their evolutionary history or by comparing their sequence similarity, can be a crucial step in hypothesis testing. Interpreting relationships of human immunodeficiency virus type 1 (HIV-1) sequences can be challenging because of their rapidly evolving genomes, but it may also lead to a better understanding of the underlying biology. Several studies have focused on the evolution of HIV-1, but there is little information to link sequence similarities and evolutionary histories of HIV-1 to the epidemiological information of the infected individual. Our goal was to correlate patterns of HIV-1 genetic diversity with epidemiological information, including risk and demographic factors. These correlations were then used to predict epidemiological information through analyzing short stretches of HIV-1 sequence. Using standard phylogenetic and phenetic techniques on 100 HIV-1 subtype B sequences, we were able to show some correlation between the viral sequences and the geographic area of infection and the risk of men who engage in sex with men. To help identify more subtle relationships between the viral sequences, the method of multidimensional scaling (MDS) was performed. That method identified statistically significant correlations between the viral sequences and the risk factors of men who engage in sex with men and individuals who engage in sex with injection drug users or use injection drugs themselves. Using tree construction, MDS, and newly developed likelihood assignment methods on the original 100 samples we sequenced, and also on a set of blinded samples, we were able to predict demographic/risk group membership at a rate statistically better than by chance alone. Such methods may make it possible to identify viral variants belonging to specific demographic groups by examining only a small portion of the HIV-1 genome. Such predictions of demographic epidemiology based on sequence information may become valuable in assigning different treatment regimens to infected individuals.</p>
<!-- suppData -->

<h1>Supplemental Data</h1>
<p>Additional material from this study is available on the journal web
site:</p>
<ul>
<li><a href="http://mbe.oxfordjournals.org/cgi/data/20/7/1168/DC1/1">Supplementary MANOVA
analysis</a></li>
<li><a href="http://mbe.oxfordjournals.org/cgi/data/20/7/1168/DC1/2">Supplementary
Figure 6</a> &ndash;
color version of Figure 6, Phylogenetic relationship of the 100
HIV-1 subtype B sequences</li>
</ul>

</div>


<div class="clear"></div>
</article>

</section>

</div>
</body>
<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/640-Predicting-demographic-group-structures-based-on-DNA-sequence-data by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:08:03 GMT -->
</html>