<!DOCTYPE html>
<html>

<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/714-Association-study-of-common-genetic-variants-and-HIV-1-acquisition-in-6-300-infected-cases-and-7-200-controls by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:05:54 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    
<title>Association study of common genetic variants and HIV-1 acquisition in 6,300 infected cases and 7,200 controls. | Mullins Molecular Retrovirology Lab</title>

    
    <link rel="stylesheet" href="style.css" />
    
    <link rel="alternate" type="application/atom+xml" title="Atom feed" href="feed" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-136625084-1"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-136625084-1');
    </script>
</head>
<body id="entry">

<div id="container">

<header>

<h1>Mullins Molecular Retrovirology Lab</h1>

<div class="description">
<ul>
<li>Department of Microbiology</li>
<li>School of Medicine</li>
<li>University of Washington</li>
</ul>

</div>


</header>

<section id="nav">
    <nav>
        
        <ul>
            <li class=""><a href="../index.html">Home</a></li>
            <li class=""><a href="../research/index.html">Research</a></li>
            <li class="here"><a href="index.html">Publications</a><ul><li><a href="supplemental.html">Supplemental Data</a></li></ul></li>
            <li class=""><a href="../protocols/index.html">Protocols</a></li>
            <li class=""><a href="../people/index.html">People</a></li>
            <li class=""><a href="../contact/index.html">Contact</a></li>
            <li class=""><a href="../help/index.html">Help</a></li>
        </ul>
    </nav>
</section>

<section id="links">
    <nav id="quicklinks">
        <h2>Software &amp; Tools</h2>

        <ul>
            <li><a href="http://indra.mullins.microbiol.washington.edu/">Bioinformatics Tools</a></li>
            <li><a href="https://github.com/mullinslab">GitHub</a></li>
            <li><a href="../hiris/index.html">HIRIS</a></li>
            <li><a href="../HMA/index.html">HMA Subtyping Kit</a></li>
            <li><a href="../isdb/index.html">ISDB</a></li>
            <li><a href="../tcozy/index.html">TCozy</a></li>
            <li><a href="https://viroverse.washington.edu/">Viroverse</a></li>
        </ul>
    </nav>

    <nav id="lablinks">
        <h2>Lab Links</h2>

        <ul>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/pub/doc/">Computing Docs</a></li>
            <li><a href="../hiris/private/index.html">HIRIS (Private)</a></li>
            <li><a href="https://tcozy.mullins.microbiol.washington.edu/ice-floe/">ICE Floe</a></li>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/git/">Internal Git</a></li>
            <li><a href="../twiki/index.html">Internal wiki</a></li>
            <li><a href="https://viroverse.washington.edu/viroblast/">Local ViroBLAST</a></li>
            <li><a href="https://redash.mullins.microbiol.washington.edu/">Redash</a></li>
            <li><a href="https://mullinslab.slack.com/">Slack</a></li>
            <li><a href="../support/index.html">Ticketing system</a></li>
        </ul>
    </nav>

</section>

<div id="cfar"><a href="http://cfar.washington.edu/">
    <img src="../static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg" width="500" height="157" srcset="/static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg 1x, /static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_1000x314.jpg 2x" alt="University of Washington/Fred Hutch Center for AIDS Research">
</a></div>

<section id="content">
    
<!-- source file: content/publications/mclaren_2013_plos_pathogens-550.md -->
<article>

<div id="intro">
    <h1>Citation Information</h1>
<p>McLaren PJ, Coulonges C, Ripke S, van den Berg L, Buchbinder S, Carrington M, Cossarizza A, Dalmau J, Deeks SG, Delaneau O, De Luca A, Goedert JJ, Haas D, Herbeck JT, Kathiresan S, Kirk GD, Lambotte O, Luo M, Mallal S, van Manen D, Martinez-Picado J, Meyer L, Miro JM, Mullins JI, Obel N, O&#39;Brien SJ, Pereyra F, Plummer FA, Poli G, Qi Y, Rucart P, Sandhu MS, Shea PR, Schuitemaker H, Theodorou I, Vannberg F, Veldink J, Walker BD, Weintrob A, Winkler CA, Wolinsky S, Telenti A, Goldstein DB, de Bakker PI, Zagury JF, Fellay J (2013). <span class="title">Association study of common genetic variants and HIV-1 acquisition in 6,300 infected cases and 7,200 controls.</span> <cite class="journalName">PLoS pathogens</cite>, <span class="journalIssue">9(7)</span>, e1003515. (<a href="http://ncbi.nlm.nih.gov/pubmed/23935489">pubmed</a>) (<a href="https://doi.org/10.1371/journal.ppat.1003515">doi</a>)</p>
</div>


<div id="more">
    <h1>Abstract</h1>
<p>Multiple genome-wide association studies (GWAS) have been performed in HIV-1 infected individuals, identifying common genetic influences on viral control and disease course. Similarly, common genetic correlates of acquisition of HIV-1 after exposure have been interrogated using GWAS, although in generally small samples. Under the auspices of the International Collaboration for the Genomics of HIV, we have combined the genome-wide single nucleotide polymorphism (SNP) data collected by 25 cohorts, studies, or institutions on HIV-1 infected individuals and compared them to carefully matched population-level data sets (a list of all collaborators appears in Note S1 in Text S1). After imputation using the 1,000 Genomes Project reference panel, we tested approximately 8 million common DNA variants (SNPs and indels) for association with HIV-1 acquisition in 6,334 infected patients and 7,247 population samples of European ancestry. Initial association testing identified the SNP rs4418214, the C allele of which is known to tag the HLA-B<em>57:01 and B</em>27:05 alleles, as genome-wide significant (p = 3.6 × 10⁻¹¹). However, restricting analysis to individuals with a known date of seroconversion suggested that this association was due to the frailty bias in studies of lethal diseases. Further analyses including testing recessive genetic models, testing for bulk effects of non-genome-wide significant variants, stratifying by sexual or parenteral transmission risk and testing previously reported associations showed no evidence for genetic influence on HIV-1 acquisition (with the exception of CCR5Δ32 homozygosity). Thus, these data suggest that genetic influences on HIV acquisition are either rare or have smaller effects than can be detected by this sample size.</p>
</div>


<div class="clear"></div>
</article>

</section>

</div>
</body>
<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/714-Association-study-of-common-genetic-variants-and-HIV-1-acquisition-in-6-300-infected-cases-and-7-200-controls by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:05:54 GMT -->
</html>