<!DOCTYPE html>
<html>

<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/978-Demographic-processes-affect-HIV-1-evolution-in-primary-infection-before-the-onset-of-selective-processes by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:06:09 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    
<title>Demographic processes affect HIV-1 evolution in primary infection before the onset of selective processes. | Mullins Molecular Retrovirology Lab</title>

    
    <link rel="stylesheet" href="style.css" />
    
    <link rel="alternate" type="application/atom+xml" title="Atom feed" href="feed" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-136625084-1"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-136625084-1');
    </script>
</head>
<body id="entry">

<div id="container">

<header>

<h1>Mullins Molecular Retrovirology Lab</h1>

<div class="description">
<ul>
<li>Department of Microbiology</li>
<li>School of Medicine</li>
<li>University of Washington</li>
</ul>

</div>


</header>

<section id="nav">
    <nav>
        
        <ul>
            <li class=""><a href="../index.html">Home</a></li>
            <li class=""><a href="../research/index.html">Research</a></li>
            <li class="here"><a href="index.html">Publications</a><ul><li><a href="supplemental.html">Supplemental Data</a></li></ul></li>
            <li class=""><a href="../protocols/index.html">Protocols</a></li>
            <li class=""><a href="../people/index.html">People</a></li>
            <li class=""><a href="../contact/index.html">Contact</a></li>
            <li class=""><a href="../help/index.html">Help</a></li>
        </ul>
    </nav>
</section>

<section id="links">
    <nav id="quicklinks">
        <h2>Software &amp; Tools</h2>

        <ul>
            <li><a href="http://indra.mullins.microbiol.washington.edu/">Bioinformatics Tools</a></li>
            <li><a href="https://github.com/mullinslab">GitHub</a></li>
            <li><a href="../hiris/index.html">HIRIS</a></li>
            <li><a href="../HMA/index.html">HMA Subtyping Kit</a></li>
            <li><a href="../isdb/index.html">ISDB</a></li>
            <li><a href="../tcozy/index.html">TCozy</a></li>
            <li><a href="https://viroverse.washington.edu/">Viroverse</a></li>
        </ul>
    </nav>

    <nav id="lablinks">
        <h2>Lab Links</h2>

        <ul>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/pub/doc/">Computing Docs</a></li>
            <li><a href="../hiris/private/index.html">HIRIS (Private)</a></li>
            <li><a href="https://tcozy.mullins.microbiol.washington.edu/ice-floe/">ICE Floe</a></li>
            <li><a href="https://hercules.mullins.microbiol.washington.edu/git/">Internal Git</a></li>
            <li><a href="../twiki/index.html">Internal wiki</a></li>
            <li><a href="https://viroverse.washington.edu/viroblast/">Local ViroBLAST</a></li>
            <li><a href="https://redash.mullins.microbiol.washington.edu/">Redash</a></li>
            <li><a href="https://mullinslab.slack.com/">Slack</a></li>
            <li><a href="../support/index.html">Ticketing system</a></li>
        </ul>
    </nav>

</section>

<div id="cfar"><a href="http://cfar.washington.edu/">
    <img src="../static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg" width="500" height="157" srcset="/static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_500x157.jpg 1x, /static/_img/2c/47ca/CFAR-logo-rev-10-22-18_4aa6fe47bc_1000x314.jpg 2x" alt="University of Washington/Fred Hutch Center for AIDS Research">
</a></div>

<section id="content">
    
<!-- source file: content/publications/herbeck_2011/herbeck_2011-488.md -->
<article>

<div id="intro">
    <h1>Citation Information</h1>
<p>Herbeck JT, Rolland M, Liu Y, McLaughlin S, McNevin J, Zhao H, Wong K, Stoddard JN, Raugi D, Sorensen S, Genowati I, Birditt B, McKay A, Diem K, Maust BS, Deng W, Collier AC, Stekler JD, McElrath MJ, Mullins JI (2011). <span class="title">Demographic processes affect HIV-1 evolution in primary infection before the onset of selective processes.</span> <cite class="journalName">Journal of virology</cite>, <span class="journalIssue">85(15)</span>, 7523-34. (<a href="http://ncbi.nlm.nih.gov/pubmed/21593162">pubmed</a>) (<a href="https://doi.org/10.1128/JVI.02697-10">doi</a>)</p>
</div>


<div id="more">
    <h1>Abstract</h1>
<p>HIV-1 transmission and viral evolution in the first year of infection were studied in 11 individuals representing four transmitter-recipient pairs and three independent seroconverters. Nine of these individuals were enrolled during acute infection; all were men who have sex with men (MSM) infected with HIV-1 subtype B. A total of 475 nearly full-length HIV-1 genome sequences were generated, representing on average 10 genomes per specimen at 2 to 12 visits over the first year of infection. Single founding variants with nearly homogeneous viral populations were detected in eight of the nine individuals who were enrolled during acute HIV-1 infection. Restriction to a single founder variant was not due to a lack of diversity in the transmitter as homogeneous populations were found in recipients from transmitters with chronic infection. Mutational patterns indicative of rapid viral population growth dominated during the first 5 weeks of infection and included a slight contraction of viral genetic diversity over the first 20 to 40 days. Subsequently, selection dominated, most markedly in env and nef. Mutants were detected in the first week and became consensus as early as day 21 after the onset of symptoms of primary HIV infection. We found multiple indications of cytotoxic T lymphocyte (CTL) escape mutations while reversions appeared limited. Putative escape mutations were often rapidly replaced with mutually exclusive mutations nearby, indicating the existence of a maturational escape process, possibly in adaptation to viral fitness constraints or to immune responses against new variants. We showed that establishment of HIV-1 infection is likely due to a biological mechanism that restricts transmission rather than to early adaptive evolution during acute infection. Furthermore, the diversity of HIV strains coupled with complex and individual-specific patterns of CTL escape did not reveal shared sequence characteristics of acute infection that could be harnessed for vaccine design.</p>
<!-- suppData -->

<h1>Supplemental Data</h1>
<p><a href="../static/_img/7d/42b4/SupplementalFigure1_1a8096f5ea.tiff">SupplementalFigure1.tif</a>
InSites diagrams of genome sequences for transmission pair
4</p><p><a href="../static/_img/23/c1a8/SupplementalFigure4_0bee530084.tiff">SupplementalFigure4.tif</a>
InSites diagrams of genome sequences for transmission pair 3</p><p>Alignment of phylogenetically-informative sites identified in whole
proteome sequences relative to the visit 1 consensus sequence in the
seroconverter. Header row with visit 1 consensus sequence with HXB2
numbering is shaded in grey for positively selected sites (as detected
by FEL or by a simulation approach (54, 42)) and purple for putative
N-linked glycosylation sites. Footer row represents known or predicted
epitopes: AA sites within epitopes are shaded in black, AA sites located
near known or predicted epitopes (up to 5 AA away) are shaded in grey.
Green boxes surround HIV-1 segments recognized by IFN-Î³ ELISpot
responses. Red boxes surround mutually exclusive mutation patterns.
Orange cells represent forward mutations (decrease in database frequency
of the AA by 50% or more), blue cells represent reverse mutations
(increase in database frequency of the AA by 50% or more), green cells
represent less substantial changes in database frequency. In addition,
the upper part of the diagram corresponds to sequences from the
transmitter, and sequences are also compared to the visit 1 consensus in
the seroconverter; days post symptoms are displayed on the left of each
row. Sites that differed between transmitter and seroconverter are
highlighted. AA sites are highlighted in pink when the AA in the
seroconverter was different from the AA found in the transmitter and the
AA in the transmitter has a higher database frequency than the AA found
in the seroconverter (by 50% or more). AA sites were highlighted in blue
when the AA in the seroconverter is different from the AA found in the
transmitter and the AA in the transmitter has a lower database frequency
than the AA found in the seroconverter (by 50% or
more).</p><p><a href="../static/_img/67/fe60/SupplementalFigure2_1a889f94ee.tiff">SupplementalFigure2.tif</a></p><p>Dips in genetic diversity and APOBEC3F/G-induced mutations. The mean
pairwise nucleotide diversity across genomes (corrected with the HKY
substitution model) is represented by a black line. The mean number of
APOBEC3G/F-mediated mutations is represented by a blue line; a dotted
black line corresponds to the mean number of control mutations (as
measured using Hypermut
2.0).</p><p><a href="../static/_img/65/c65c/SupplementalFigure3_f3cebdb009.tiff">SupplementalFigure3.tif</a></p><p>InSites diagram of genome sequences for subject S3. Alignment of
phylogenetically-informative sites identified in whole proteome coding
sequences relative to the visit 1 consensus sequence in the
seroconverter. Figure is as described in the legend for Supplemental
Figures 1 and 4.</p>
</div>


<div class="clear"></div>
</article>

</section>

</div>
</body>
<!-- Mirrored from mullinslab.microbiol.washington.edu/publications/978-Demographic-processes-affect-HIV-1-evolution-in-primary-infection-before-the-onset-of-selective-processes by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 21 Jul 2025 18:06:17 GMT -->
</html>